\chapter{Investigations of physiological remodeling upon a nitrogen 
upshift}
\label{chapter:four}

This chapter describes lines of investigation that were not
pursued deeply, but may inform future investigations of the
physiological remodeling that occurs as yeast resumes rapid
growth.

The study of microbial physiology is a long standing area of
investigation, and with modern systems biology approaches this
question of how the physiological composition of microbes change in
order to accomplish the essential project of growth is still the
subject of advances both quantitative and conceptual 
\parencite{slator1918some,henrici1928morphologic,schaechter1958dependency,kjeldgaard1958transition,wehr1969macromolecular,waldron1977synthesis,carter1978protein,waldron1975effect,kief1981coordinate,scott2010interdependence,erickson2017global,kafri2016cost,metzl2017principles}.
In recent years, the study of changes in abundance of specific mRNA
factors in the budding yeast has characterized a phenomenon in which
approximately one quarter of the yeast transcriptome scales with
growth rate \parencite{brauer2008coordination,airoldi2009predicting}.
This phenomenon is characterized at the level of molecular species,
and thus can be compared to changes that occur in response to
stressors, summarized as the Environmental Stress Response 
\parencite{gasch2000genomic}. 
In addition, a shared signature of knockout mutants, commonly used to
probe gene function, is that associated with changes in cell-cycle
progression distribution due to growth rate changes
\parencite{o2014cell}.
Thus the appreciation of the systematic physiological changes that
occur in response to genetic perturbations holds light to many
biological problems, even if only to identify the domineering and 
confounding factor of growth-associated physiological changes.

%
%These, and other observations, described microbial growth generally as 
%possessing a kind of "interia" [@henrici1928morphologic], where the
%growth rate 
%of the old culture is maintained for a short period after a
%nutrient-shift.
%@sherman1924function studied changes in resistance in bacterial
%salt-stress 
%during this lag-phase to conclude that old cells would remodel their
%physiological state prior to initiating growth at the maximal rate
%possible of the new environment.
%@kjeldgaard1958transition, @wehr1969macromolecular
%waldron1977synthesis  found that following a nitrogen-source upshift,
%a yeast culture will continue its rate of accumulation of optical
%density
%for about 2 hours before increasing the rate to that appropriate to
%the new
%media.
%In contrast, the RNA accumulation appeared to lag only 10 minutes
%before
%accumulating at a rate temporarily faster than the new steady-state
%rate.
%This suggests that rRNA, thus ribosome content, 
%is a leading feature of this physioloical remodeling.
%A carbon-source upshift also results in massive regulation 
%that reprograms the yeast physiology and transcriptome (and other
%-omes) for 
%rapid growth \cite{kief1981coordinate}.
%It was originally observed that yeast paradoxically halts growth
%upon a glucose up-shift for about 60 minutes before resuming
%growth and protein-synthesis [@kief1981coordinate]. However,
%ribosomal RNA and protein were shown to still rapidly increase
%across the span of an hour, even while the rest of the cellular
%growth and division are halted. It is important to note that
%this study measured nascent relative to extant - that is, an
%increase in the ratio of nascent to extant can result from either
%accelerated synthesis or degradation of extant transcripts 
%[@kief1981coordinate].

%
%
%
\section{Changes in poly-adenylated transcript content per cell 
upon changes in growth rates}
%
%
%

This section describes work that contributed to a submitted article 
%that 
%has been submitted, rejected, and is currently undergoing revision 
%of the text to address a particular biological question with more
%focus.
%The article was 
titled:
\textit{"Growth Rate-Dependent Global Amplification of Gene Expression."}
Authorship of this article is: 
Niki Athanasiadou, Benjamin Neymotin, Nathan Brandt, 
\textbf{Darach Miller}, Daniel Tranchina, and David Gresham.
The \textit{biorxiv} draft is at \url{doi.org/10.1101/044735}

The writing and figures of this chapter are original to this document.

%
%
%
\subsection{Introduction}
%
%
%


We know that the total RNA content of a cell changes upon changes in
growth rates \parencite{waldron1975effect}.
We know that specific mRNA, each a small component of the cell's 
total RNA also change in relative abundance. A less-characterized 
question is if the whole mRNA transcriptome changes
and if this has a significant effect on the regulatory role
of absolute or relative changes in mRNA abundance.
Transcriptomic measurements are usually normalized to relative
measures, and is thus based (sometimes explicitly
\parencite{love2014moderated})
on the assumption that the total transcriptome does not change in 
abundance.  However, we now know of cases of where this 
assumption is violated \parencite{nie2012c}.

Spike-in normalized RNA sequencing can estimate absolute mRNA
abundance per cell, but has been criticized before for
being "too noisy" and instead computational methods of "removing
unwanted variation" were used \parencite{risso2014normalization}. 
Led by Rodoniki Athansidou, our group pursued a more thorough approach
to this design by normalizing RNA sequencing data using the ERCC
spike-in set \parencite{jiang2011synthetic}, 
using preliminary sequencing runs
to first determine the appropriate amount of spike-ins necessary for
accurate sequencing. Then, yeast were grown in systematically varied
nutrient limitations of growth, then RNA sequencing using a known
quantity of the exogenous spike-ins was used to normalize the
measurements to absolute mRNA per cell.

I sought to complement this work by orthogonally estimating the size 
of the whole yeast transcriptome. To do this, I adapted the screening 
strategy of \cite{amberg1992isolation} to flow cytometry. 
Essentially, this utilized a poly-deoxythymidine oligo singly labeled
with a fluorophore. This was hybridized in with the fixed and
permeabilized yeast cell, and the resulting fluorescence after washing
is taking to be a proxy for the number of hybridized poly-dT probes,
presumably hybridized to a poly-adenosine sequence, and thus mRNA.

Another motivation of this was to serve as a fixation-digestion
control for methods involving single-gene mRNA FISH.
We had patterns of mRNA FISH hybridization signal that appeared
bimodal \autoref{fig:gap1Delete}. 
This could be a technical issue of incomplete permeabilization due to
over-fixing, or a biological phenomenon.
To distinguish the two would take two-color FISH, with a positive
control \parencite{andersen2014genetic}.
Since nitrogen-limitation causes a severe restriction of the total
transcriptome content, we don't have an obvious pick for a uniformly
expressed positive control. 
However, most of the mRNA should be poly-adenylated, so FISH against
that sequence should be present in all cells, and in high-copy.
While I did not integrate this into the single gene mRNA FISH as an
internal control, I did use it to optimize fixation/permeabilization 
conditions.

%
%
%
\subsection{The assay design}
%
%
%

The assay uses a similar fixation permeabilization method as the
single-gene mRNA FISH assay, then an overnight hybridization using
dextran sulfate against a poly-dT probe, then flow cytometry.

The yeast cells are sampled via vacuum filtration onto nylon
filters, then the filters are quickly flash-frozen in liquid nitrogen.
These are resuspended in 0.75x PBS buffered 4\% PFA (from ampules from
EMS), the cells vortexed off the filter, then the filter discarded.
The cell suspension in the fixative is incubated for hours at RT to
complete fixation, with the assumption that rapid fixation halts
RNA metabolism in the cell and long-term fixation stabilizes the fixed
components into a configuration that can survive digestion and 
permeabilization. The fix is critically quenched using 2.5M glycine,
then collected by centrifugation and washed with PBS. The cells are
digested for one hour at 37C using lyticase and beta-mercaptoethanol
in 1.2M sorbitol buffered
by potassium phosphate at about 7.4 pH, with 20mM vanadyl
ribonucleoside complex to inhibit RNAses. 
This is washed and further permeabilized with 70\% ethanol overnight,
then is resuspened using hybridization buffer
(10\% dextran sulfate w/v, 2x SSC final, 100ug/ml ecoli tRNA, 
250mg/ml salmon sperm DNA) plus 100nM of a (dT)50+V oligo 5'-labeled
with with Alexa 488, as ordered from IDT. 
This is incubated for 14+ hours on a 37C roller drum, then washed with
2x SSC several times before resuspending in PBS and flowing through an
Accurri flow cytometer.
Poly(A) content signal was determined by the signal area on the
514/20nm detector.

To test this procedure, I used RNAseA treated cells as a negative
control.
\autoref{fig:ypdnlim} shows the RNAseA-treated controls for two
samples, where the treatment abrogates the signal for the vast
majority of the cells in the sample.

To optimize this design, I varied formamide from 0\% to 50\%, and
probe concentrations from 10nM to 1$\mu$M. I found that 100nM and 0\%
formamide saturated the signal of YPD-grown cells without largely
increasing the signal on the RNAseA-treated cells.
This assay takes approximately 4 hours of work spread over 3 days.
More detailed protocol is maintained by the Gresham laboratory.

%%%%% PUT A CONTROL FIG HERE.

%
%
%
\subsection{Nutrient limitation and transcriptome size}
%
%
%

Yeast growing in
YPD complete a division approximately every 1.5 hours (0.45 specific
growth rate), while 
proline-limited media (NLimPro) only supports division approximately
every 4.5 hours (.15 specific growth rate). 
Using this poly-dT FISH method, we see differences in the total 
poly-adenylated mRNA signal between the different media conditions
(\autoref{fig:ypdnlim}).
The distributions are significantly
different (KS test and Wilcoxon, p-value < $2.2 \times 10^{-16}$).
We know that fast growing cells (YPD) have more RNA per cell, so it
appears that part of this difference is contributed by a global
scaling of the mRNA content as well.
The fold-changes in the mean and median of the YPD-grown cells versus
the proline-limited cells were 3.34 and 3.68, respectively.

\afig{
  \includegraphics[width=.7\textwidth]{img/polya_ypdnlim_controls.png}
  }{
  Wild-type yeast grown in proline-limited media (left) or YPD rich
  media (right) were assayed in exponential growth for poly-A content.
  Included are RNAsed controls treated with RNAseA, to show negative
  samples.
  The plot is cropped from 0 to $10^5$ arbitrary units of polyA
  signal to show the center of the distributions.
  The distributions from different media have different means by 
  KS or Wilcoxon tests, with unreasonably small p-values.
  \label{fig:ypdnlim}
  }{Changes in whole cell polyA content in YPD or nitrogen-limitation.}

To investigate the dynamics of changes in poly-A abundance between 
different growth conditions, I grew cells in proline-limited media
overnight to reach a steady-state of growth, then collected samples
during a nitrogen-upshift.
I assayed the poly-A content of the cells using the above assay
(\autoref{fig:upshift}).
I found that the total poly-A content took about two hours to increase
to the new steady-state of a larger transcriptome, a similar timescale
as the changes in cell size and lag in population growth rate
(\autoref{fig:figure1a}).
The final steady-state differences were of a fold-change of 2.16 and
1.93 for the mean and median poly-A content, consistent with the
change between specific growth rates of 0.15 and 0.35 being lower
than the difference with YPD (0.45 specific growth rate). 

\afig{
  \includegraphics[width=\textwidth]{img/polya_upshift.png}
  }{
  Wild-type yeast were grown in proline-limited media, then glutamine
  was added at time 0 minutes. Samples were assayed for polyA content
  using the poly-dT assay.
  \label{fig:upshift}
  }{Changes in polyA content upon a nitrogen upshift.}

Previously, others in the lab (as described at the beginning of this
section) had used ERCC-normalized RNA sequencing
to assay the absolute abundance of mRNA in yeast grown at
systematically varied growth rates (0.12, 0.2, 0.3 specific growth 
rate) in chemostats. In a repeat experiment of this, I took samples 
from chemostats limited by nitrogen or carbon at these growth rates, 
and processed them to assay the distribution of poly-A content of the 
cells. \autoref{fig:nikis} shows the distributions and the
relationship between the distribution means and the estimates from
SPARQ (the spike-in normalized RNAseq method). We see that the poly-dT
method also captures the scaling of the whole yeast transcriptome
across different growth rates, and correlates well with the spike-in
normalized method.

\afig{
  \includegraphics[width=.48\textwidth]{img/polya_niki_box.png}
  \includegraphics[width=.48\textwidth]{img/polya_niki_summary.png}
  }{
  (Left) PolyA content was estimated for cultures grown in two nutrient
  limitations at three different dilution (growth) rates.
  (Right) Comparing these measurements to SPARQ (the spike-in 
  normalized RNAseq method) shows that two methods are well 
  correlated (Pearson's r=0.95, \texttt{cor.test} p-value = 0.003684,
  dashed-line shows linear regression through all points),
  although the poly-dT method remains uncalibrated.
  \label{fig:nikis}
  }{Measuring polyA content across systematically varied growth rates
    in chemostats, and comparison to a spike-in normalized RNA 
    sequencing method.}

\subsection{Conclusion and future directions}

This assay appears to detect changes in the scaling of the yeast
mRNA content between different growth rates. 
It is consistent with spike-in normalized RNAseq (random 
hexamer-primed) estimates of the total mRNA content.
As a flow cytometry 
assay this has the potential to be used as a marker for
high-throughput investigations of the genetics of transcriptome size
changes (or regulation), using methods as described in
\nameref{subsection:bff}. 
This method offers a conveniently high-throughput assay for total
transcriptome size, and as such is one more tool that microbial
physiologists can use to probe the functional changes that occur as
organisms systemically adapt to their environments and growth
programs.

However, more work remains to use the assay to reliably inform on
these changes without incorporating an orthogonal measure.
Changes in polyA tail length could hypothetically affect
hybridization, and a distribution shifting such that more of the
functional mRNA have a tail length less than minimum tail length
requisite for hybridization would produce a similar graded effect.
Hybridization of this probe to synthetic mRNA cross-linked to a nylon
substrate would allow quantitative testing of this in similar
conditions as the hybridization occurs, provided a method for
manufacturing accurately generated poly-A tail lengths exists.

Future investigations of mRNA content per cell will illuminate
the role or significance of total mRNA abundance 
versus relative mRNA abundance in gene regulation and physiological
adjustments to changing environments. Adjustment is apparent during a 
nitrogen upshift, what causes it, and is it adaptive?

\section{Screening for genes important for remodeling physiology for
growth}

\subsection{Introduction}



With changing physiology in response to growth rate changes, many
molecular and functional phenotypes change. One of these is the
resistance to stress. 
It has been long known that slow growing cells are more resistant to
stressors
\parencite{sherman1923physiological,elliott1993stress,lu2009slow}.
Yeast appears to have adapted to its ecological niche by adopting a
boom/bust, feast or famine approach to quickly
growing during favorable conditions at the expense of stress
resistance. 
Resistance to stress seems to offer "cross-protection", and the
anti-correlation of growth rate and stress resistance suggests that
the two processes might be opposed in mechanisms to
achieve these objectives.
The dimension of coordinated cellular growth may be a simple axis that
explains much of the variation in gene expression and phenotypic
differences in budding yeast
\parencite{brauer2008coordination,lu2009slow}.

One approach to identify the characteristics required for yeast to 
achieve a faster growth rate is to monitor the regulated changes that
occur upon the upshift. We could infer
that since the most logical response to a stress is to express this
adaptation, then the gene expression increasing upon a stress must be
adaptive \parencite{gould1979spandrels}. 
This has been demonstrated to be a false assumption, at least for the
case of heatshocks, as the genes whose expression increases do not 
overlap well with the
genes important for resistance \parencite{gibney2013yeast}.
The later functional genetic measurement is possible to do in 
high-throughput, as
the yeast community has access to a yeast deletion collection and
high-throughput means of assaying genetic effects on the quantitative
phenotype of continued existence. 
Thus, an assay of the functional consequences is a more direct
approach to understand these processes.

The nitrogen upshift enriches for %offers an obvious process to enrich for
differences in growth rate, by growth. 
Subtle effects can be magnified over time, for example a
1\% growth rate defect over 7 hours would be magnified to an abundance
change of at least 20\%. However, the phenotype I am interested in is
in the completion of remodeling for rapid growth, so I am most
interested in the duration of the lag between nitrogen addition and
increased growth. Thus, the compounding of growth rate effect does not 
apply. One approach would be to repeat the upshift many times on the
same batch of cells, but this greatly confounds the fitness between 
various growth stages and does not offer the reproducibility of cells
being in a particular physiological status --- nitrogen-limitation can
take hours to reach a steady-state of signalling
\parencite{tate2013five}, and the life history of an individual cell
could have physiological consequences.

After practicing the Feynmen method with a scientific advisor
(\nameref{section:acknow}), and given
the opportunity to work with a talented young scientist named Stephen
Nyarko, we decided to pursue this question by using the correlated
phenotypes of growth and susceptibility to stress.
The logic is that if we are interested in isolating mutants that are
defective in increasing their growth rate upon a nitrogen upshift, and
an increase in growth is associated with a susceptibility to stress,
then a somewhat-lethal stress should enrich for mutants defective in
susceptibility to the stress --- ie defective in rapidly increasing 
growth rate.
Upon further reading, we found that the group of Johan Theiveilen had
used a similar approach to isolate mutants defective for increasing
growth upon repletion of glucose, and had identified new critical
components of the PKA pathway, \textit{CYR1} and \textit{GPR1}
\parencite{van2000characterization}.
Thus encouraged, we intended to use the anti-correlation of growth
and stress resistance to isolate mutants defective in resuming growth
rapidly.

\subsection{Results}

We first determined if the heatshock resistance of the wild-type
FY4 yeast changed during a nitrogen upshift.
I grew cells in proline-limited media (approximately 4.5 hour doubling
time), then added glutamine to induce the nitrogen upshift.
For each sample, cells were subject to a 52C heatshock for 30 minutes
by the addition of pre-warmed media, or for negative control were
simply kept at room temperature. The processed samples were arrayed 
in a 96 well plate, then pinned onto YPD, grown at 30C for
approximately 40 hours, and imaged (\autoref{fig:dme180}).

\afig{
  \hfill
  \includegraphics[width=.2\textwidth]{img/160328exp180sample1.png}
  \includegraphics[width=.2\textwidth]{img/160328exp180sample2.png} 
  \includegraphics[width=.2\textwidth]{img/160328exp180sample3.png}
  \includegraphics[width=.2\textwidth]{img/160328exp180sample4.png}
  \\
  \begin{tikzpicture}[overlay
      ,font=\small,%font=\ttfamily
      ,inner sep=0pt,outer sep=0pt
      ,shift={(-8,0)}]
      ]
    \node[align=left] at (0,0) (o) {};
%
    \node[align=left] at ($(o)+(5,0)$) (dil) {Culture dilutions (5-fold)};
    \draw[->] (dil) to ($(dil)+(10,0)$);
%
    \node[align=left,anchor=east] at ($(o)+(2,3.00)$) {\tiny Heatshocked};
    \node[align=left,anchor=east] at ($(o)+(2,2.65)$) {\tiny Negative};
    \node[align=left,anchor=east] at ($(o)+(2,2.30)$) {\tiny Heatshocked};
    \node[align=left,anchor=east] at ($(o)+(2,1.95)$) {\tiny Negative};
    \node[align=left,anchor=east] at ($(o)+(2,1.60)$) {\tiny Heatshocked};
    \node[align=left,anchor=east] at ($(o)+(2,1.25)$) {\tiny Negative};
%
    \node[align=left] at ($(o)+(4,4.1)$) {Before upshift};
    \node[align=left] at ($(o)+(7.8,4.4)$) {61 minutes\\after upshift};
    \node[align=left] at ($(o)+(11.2,4.4)$) {132 minutes\\after upshift};
    \node[align=left] at ($(o)+(14.6,4.4)$) {204 minutes\\after upshift};
  \end{tikzpicture}
  }{
    Wild-type (FY4) cells were subject to nitrogen-limitation, then
    a nitrogen upshift with 400$\mu$M glutamine. Cultures were
    heatshocked at 52C for 30 minutes, or at roomtemperature 
    (negative). 5-fold dilutions were plated on YPD plates.
    \label{fig:dme180}
  }{Glutamine upshift causes a lost in heatshock-resistance.}

Thus, the glutamine upshift triggers a loss in resistance to the
heatshock. We then devised a screen, wherein a barcoded 
and pooled yeast deletion collection
is grown in conditions of nitrogen-limitation then upshifted.
Samples were taking before or 
after 120 minutes after the glutamine upshift,
heatshocked or not (negatives),
then outgrown to enrich for living mutants.
These libraries were sequenced using an amplicon-sequencing procedure
to quantify the mutants in the resulting library.

Direct measurement of mutant abundance is preferred, but we used
outgrowth of the heatshocked population, counting on the severe
selection of a heatshock to appropriately select.
We did this in six biological replicates in order to generate robust
signal.
These were extracted with standard Hoffman-Winston DNA preparations,
then amplified using the same primers and protocol as described in
\parencite{robinson2014design}. These were sequenced along with other
samples on an Illumina MiSeq run.

Barcode sequencing, like other molecule-counting applications of
sequencing like RNAseq, is presented to the researcher as a relative 
measurement in integer quantities. 
One of the first steps in reading this data in is to look at the
distribution of counts per strain barcode identified
\autoref{fig:sneDist}.
We see that our heatshock and outgrowth has a much more profound
distribution of effects.
For which genes is this significant?


Numerous statistical approaches
exist to normalize the data for accurate detection of differential
abundance. One flexible and robust method is using the \texttt{voom}
statistical pre-processing step with \texttt{limma}. 
This calculates
the expected noise contributed by low integer count observation, but
has the advantage of converting the measurement to a "counts per
million" relative metric for normalization. 
It also has useful visualizations for characterizing the 
distribution of signal across complex experimental designs.

One other observation in \autoref{fig:sneDist} is that the histograms
show a log-normal distribution of high counts, then a long tail
downwards. 
Then, there appears to be a low distribution of single digit counts
which enter the distribution from around zero \autoref{fig:sneDistSum}.
These are believed to occur from spurious amplicon products or
software misalignment counting barcodes that do not exist.
To characterize this further, I used \texttt{limma/voom} to generate
plots of variance against abundance for different thresholds of
cutoffs based on total counts across the entire library
\autoref{fig:vooming}.
I found that a threshold of 30 counts in total across the library
was sufficient to remove these effects.

\afig{
  \includegraphics[width=.6\textwidth]{img/sne_histogram.png}
  }{
  Histograms of counts, for each mutant in each sample. Three
  histograms show the occurrences of these observations for the
  library before the upshift (top), 2 hours after adding glutamine
  (middle), and after the heatshock and outgrowth (bottom). The wider
  spreading is a good indication of complex selection of large
  effects occurring in the library.  \label{fig:sneDist}
  }{Histogram of mutant counts, within each sample.}
\afig{
  \includegraphics[width=.6\textwidth]{img/sne_histogram_sum.png}
  }{
  Histograms of counts, for each mutant, summed across all samples
  in the three treatments: before the upshift (top), 2 hours after 
  adding glutamine (middle), and after the heatshock and outgrowth 
  (bottom). We see that most features are log normally distributed,
  but some appear to be noisy counts near zero, due to unknown
  factors. 
  \label{fig:sneDistSum}
  }{Histogram of mutant counts, summed across samples.}
\afig{
  \includegraphics[width=.7\textwidth]{img/sne_voomer.png}
  }{
  Each plot shows each gene average abundance (x-axis) against its
  residual variation (y-axis), with a line smoothing the relationship
  as expected by \texttt{limma} modeling. The threshold of minimum
  total counts per feature is shown for each plot in the grey bar.
  We see that thresholding above 30 counts \textbf{(right)} gives us the 
  expected relationship, while not thresholding \textbf{(left)}
  demonstrates how lowly abundance counts behave aberrantly with
  artificially reduced variance that may confound statistical
  analyses of barcode sequencing data (\texttt{limma} uses the model 
  fit as the line). 
  \label{fig:vooming}
  }{Diagnostic \texttt{limma/voom} plots show the effects of 
    low-count barcodes in confounding the noise model.}

I used this tool's
flexible general linear modeling interface to ask how our treatment
enriched for particular mutants.
We saw no significant effects from a glutamine upshift, 
confirming the intuition that this
is such a subtle effect of momentary fitness that it becomes hard to 
detect without amplification. 
Testing for the effect of changes in abundance based on a
glutamine treatment before the heatshock, we find that four deletion 
strains significantly
(multiple-hypothesis adjusted p-values < 0.05) increase in abundance 
specifically after glutamine treatment, and 41 are decreased in
abundance.

Of the four genes increased in abundance (suggesting a failure to
resume rapid growth), \textit{SLA1} and
\textit{CAP2} are involved in actin binding and dynamics.
\textit{SLA1} is involved in assembly of the cortical actin
cytoskeleton \parencite{holtzman1993synthetic}, 
while \textit{CAP2} is an actin barbed-end
capping protein that localizes to cortical actin patches
\parencite{amatruda1990disruption}. 
This suggests that these mutants specifically are involved in
remodeling the cortical exoskeleton in a way that makes cells more
susceptible to heatshock, or that these mutants are defective in
increases in stress-resistance associated with slow growth rates.
\textit{SXM1} over-expression rescues mutants defective in mRNA export 
from the nucleus \parencite{seedorf1997importin}, suggesting that it
may play a role in mRNA export itself and that mRNA export may
regulate some important downstream factor associated with increasing
growth.
\textit{MAE1} encodes a malate dehydrogenase. This reaction takes
malate, a citric-acid cycle metabolite, and converts it to pyruvate
\parencite{boles1998identification}.
Pyruvate is an essential substrate for the
biogenesis of the carbon structures of alanine, valine, and other 
amino-acids.
Carbon-skeletons of glutamine can enter the citric-acid cycle from a
point between the entry of pyruvate, and shunt from malate to pyruvate
via \textit{MAE1}.
Considering the enrichment of pyruvate metabolism mRNA identified as 
destabilized in the 4tU label-chase work in 
\autoref{subsection:stabilityChanges}, and that the same experiment 
showed either a stabilization or dramatic synthesis up-regulation of 
\textit{MAE1} mRNA upon the nitrogen upshift, one prediction might 
be that Mae1p provides a shunt by which yeast re-directs the excess of
carbon skeletons from glutamine deamination through the citric-acid
cycle to provide substrates for alanine biosynthesis. This may be
adaptive.

To explore this, I regenerated a \textit{mae1}$\Delta$ mutant using
a KanMX knockout cassette amplified from the yeast deletion
collection, confirming incorporation by PCR. I subjected this mutant
to a glutamine upshift, and saw an increase in the lag-phase duration
(\autoref{fig:mae1})
compared to wild-type (\autoref{fig:figure1a}). I sought to test if this
was specifically due to disruption of the malate to pyruvate shunt for
the effect of alanine metabolism, and so repeated the experiment but
added 200$\mu$M alanine, 200$\mu$M pyruvate, or water (mock) to the 
cell culture at the same time as glutamine.
I did not see a significant effect on the growth rate increase
(\autoref{fig:mae1}).
Thus, disruption of this gene may not result in slower upshift in
growth by virtue of blocking this metabolic pathway, but instead the
metabolic state of the cell before the upshift may not be well 
prepared to resume rapid growth.

\afig{
  \includegraphics[width=.7\textwidth]{img/dme231.png}
  \includegraphics[width=.7\textwidth]{img/dme242.png}
  \includegraphics[width=.7\textwidth]{img/dme244.png}
  }{
  A \textit{mae1}$\Delta$ strain was subject to a glutamine upshift.
  \textbf{(Top)} The mutant alone appears to show a slight defect in the lag 
  phase (approximately 3 hours compared to approximately 2 hours
  \autoref{fig:figure1a}).
  \textbf{(Middle and bottom)} The mutant had glutamine or glutamine and either
  alanine (middle) or pyruvate (bottom) added, with two cultures per
  treatment. Neither showed a significant effect in reducing lag phase
  or increasing growth rate.
  \label{fig:mae1}
  }{A \textit{mae1}$\Delta$ mutant is slower in a glutamine upshift,
    but this is not rescued by supplementation with alanine or
    pyruvate.}

\subsection{Conclusion}

We found mutants knocked out for several non-essential genes changed
their relative susceptibility upon heatshock treatment, suggesting
that their resistance does not decrease as much as wild-type upon the
re-addition of a nitrogen source with adding glutamine.
These mutants could be involved in either the increase in stress
resistance upon slow growth conditions, or the decrease in stress
resistance upon increase in growth. 
For the factors of the actin cytoskeleton, this points towards a
hypothesis that the increase in stress resistance results from
specifically the cortical actin network, and would be testable by
determining when these mutants are more or less resistant to stress
than the wild-type. For \textit{MAE1}, I found that there appears to
be a longer lag phase, but this is not rescued by addition of pyruvate
or alanine. This suggests that the deletion of this gene puts the cell
in a metabolic configuration less capable of rapidly increasing growth
rate upon glutamine addition.

\iffalse
%
%
%
\section{Apparent cell-cycle halt upon nitrogen-upshift}
%
%
%

Apparent cell-cycle halt upon
glutamine addition Previous work on cell cycle halt, basically just
alberghina, PKA and CLN1 This phenomenon has been previously seen.
Upshift ecoli, they get bigger.  This is thought to be because the
critical cell size threshold has been reset by growth signalling
pathways to a larger size. However, this result might argue that
instead it could be regulated by CLN1 transcript abundance.
Alberghina’s demonstrated that depends on Swi4?p, so there you go

Later, it has been shown that this halt in growth seems to occur
through a PKA-mediated repression of
CLN1$\cite{jiang1998??,$tokiwa1994??}



Experiment, results Conclusion ( anything that didn’t get into chapter
3 )
\fi
