\documentclass[12pt,letterpaper]{memoir}
\usepackage{graphicx}
\usepackage[margin=1in]{geometry}

\usepackage[utf8]{inputenc}
\usepackage{DejaVuSansCondensed}
%\usepackage{helvet}
\renewcommand\familydefault{\sfdefault} 
\usepackage[T1]{fontenc}

\usepackage{wrapfig}
\usepackage{framed}
%\usepackage{subfigure}

\usepackage{pdfpages}

\usepackage{float}
\let\origfigure\figure
\let\endorigfigure\endfigure
\renewenvironment{figure}[1][2] {
  \expandafter\origfigure\expandafter[H]
  } { \endorigfigure }

\usepackage{amsmath}

\usepackage{tikz}
\usepackage{pgfmath}
%\usepackage{pgffor}
\usetikzlibrary{arrows,positioning,shapes,patterns,decorations} 
\tikzstyle{every picture}+=[remember picture]

\usepackage[backend=bibtex8%biber%bibtex8
  ,backref=true,hyperref=true
  ,refsection=none %  ,refsection=chapter
  ,maxcitenames=2
  ,style=authoryear-comp]{biblatex}
\addbibresource{references_dissertation_darachmiller.bib}

\usepackage{hyperref}
\hypersetup{colorlinks, citecolor=blue
  , filecolor=black, linkcolor=blue, urlcolor=black }

\makeatletter 
\def\makemytitle{
  \newpage
\begin{center}
\vfill
  \@title\\
\vfill
  by\\
\vfill
  \@author\\
\vfill
  A dissertation submitted in partial fulfillment \\
  of the requirements for the degree of \\
  Doctor of Philosophy \\
  Department of Biology \\
  New York University \\
\vfill
  \monthz, \yearz\\
  \end{center}
\vfill
\begin{flushright}
  $\rule{5cm}{0.01mm}$\\
  David Gresham
\end{flushright}
  \pagebreak
\begin{center}
  \textcopyright \hspace{0.2em} \@author \\
  All rights reserved, \yearz \\
\end{center}
  \pagebreak
}
\makeatother

\title{
The extent of mRNA destabilization and 
genetic factors of rapid \textit{GAP1} repression 
upon a nitrogen upshift.
}
\author{Darach Miller}
\def\monthz{May}
\def\yearz{2018}

%
%
%
%
%
%

% Memoir layout options
\setulmargins{1.0in}{*}{*}
\setheadfoot{15pt}{0.75in}
\settypeblocksize{8.25in}{*}{*}
\setlrmarginsandblock{1.0in}{1.0in}{*}
%
\openright
\chapterstyle{dash} %dash, dowding, ntglike, thatcher
\setlength\beforechapskip{0pt}%-\baselineskip}
\aliaspagestyle{cleared}{plain}
\aliaspagestyle{chapter}{plain}
%
\setlength\floatsep{0em}
\setlength\textfloatsep{0em}
\setlength\intextsep{0em}
\renewcommand{\topfraction}{0.90}
\renewcommand{\bottomfraction}{0.9}
\renewcommand{\textfraction}{0.10}
%
\setbeforesecskip{-1.0em plus -0em minus -0em}
\setbeforesubsecskip{-1.0em plus -0em minus -0em}
\setbeforesubsubsecskip{-1.0em plus -0em minus -0em}
\setaftersecskip{1em}
\setaftersubsecskip{1em}
\setaftersubsubsecskip{1em}
\setsecheadstyle{\large\bfseries\raggedright}
\setsubsecheadstyle{\normalsize\bfseries\raggedright}
\setsubsubsecheadstyle{\normalsize\bfseries\raggedright}
%
\abnormalparskip{0em}
%
\checkandfixthelayout

%\newcommand{\openrule}{
%  \par\noindent\rule{.5\textwidth}{1pt}\par\vskip-0.2em
%  \noindent\rule[\baselineskip]{1pt}{\baselineskip}}
%
%\newcommand{\closerule}{
%  \par\hfill\rule[-\baselineskip]{1pt}{\baselineskip}
%  \par\vskip-0.2em\hfill\rule{.5\textwidth}{1pt}\par}

\newcommand{\afig}[3]{
  \begin{figure}[t]
%    \vspace{-10pt}
    \begin{center}
    #1
    \end{center}
    \vspace{-10pt}
    \caption[#3]{\textbf{#3} #2}
%    \vspace{-10pt}
  \end{figure}
  }

%
\includeonly{
  dissertation_darachmiller_1,
  dissertation_darachmiller_2,
  dissertation_darachmiller_3,
  dissertation_darachmiller_4,
  dissertation_darachmiller_5
}


% hax to get the compiler to shut up, ripped from SO
\hfuzz=40pt
\vfuzz=40pt
\hbadness=5000
\vbadness=\maxdimen

\setsecnumdepth{subsubsection}
\settocdepth{subsection}

\begin{document}
\DoubleSpacing
\frontmatter
\pagestyle{empty}
%\renewcommand{\thefigure}{\arabic{figure}}

\makemytitle

\iftrue
\begin{quote}
\SingleSpace
"Science is a match that a person has just got alight. 
They thought they were in a room --- 
in moments of devotion, a temple --- 
and that this light would be reflected from and display walls 
inscribed with wonderful secrets and pillars carved with 
philosophical systems wrought into harmony. 
\vfill
It is a curious sensation, now that the preliminary splutter is 
over and the flame burns up clear, to see lit just their hands and 
just a glimpse of themselves and the patch they stand on visible, and 
around them, in place of all that comfort and beauty they
anticipated, \vspace{0.5em} darkness still."

\hfill - H.G. Wells, 1891, \\\vspace{1em}\hfill adapted for 2018
\end{quote}
\fi

\newpage

\section*{Acknowledgements}
\addcontentsline{toc}{section}{\numberline{}Acknowledgements}

\label{section:acknow}

\iffalse


The culture built over these many years at NYU Biology is a unique 
moment of growth, and these informal linkages play a surprisingly
important role in facilitating the collaboration and support
necessary.As Benjy Neymotin 
pointed out several years ago: "it takes a village".

Specifically,
I thank the kind folks on the 4th floor of CGSB, namely
the research group led by Christine Vogel. I am glad we moved to share
space and scientific discussions.
I also thank Ken Birnbaum and his group for helpful conversations and
access to the lucky FISH incubator.
Across the street, I thank Viji Subramanian, Andreas Hochwagen, 
and his research group for microscope access and tolerating my
questions.
I thank Andres Mansisidor and Matt Paul for helpful reading and 
comments on the paper, and otherwise.

Away from NYU, I thank
Evelina Tutucci for demonstrating the mRNA FISH method to me,
helping me understand just how finicky it can be.
I thank Megan McClean for openly sharing a FISH protocol on
OpenWetWare (although we've never talked).
On the west coast (and now elsewhere) I thank Carole Hom, Ian Korf, 
and Patrice Kohel at UC Davis, Nitin Baliga Aaron Brooks at ISB
Seattle, but most importantly Marc Facciotti at UC Davis for 
giving me a chance at getting into science late.

I want to thank Sarah Nguyen for keeping me together, like 

I thank Michi, Henri(ci), and Tai for making me get up and come 
home on time.


I thank Sue Miller for devising a clever genetic screen for yeast
strains that fail to re-initiate growth programs, as well as for
being so supportive through the years.
I also thank my Kendra Miller for inspiring and egging me on, 
a knowingly efficacious combination of support for folks like me.
I thank Harry Miller for support and inspiring me to wander this way,
and to never stop learning new things.

I thank staff at eBioscience/Affymetrix/ThermoFisher for support with
the Quantigene/FlowRNA probe set and patience through the years.

I thank the Cold Spring Harbor Yeast Course, especially the instructors
Maitreya Dunham, Marc Gartenberg, and Grant Brown for introducing
me to the community and (re)-lighting a fire to help me finish.
That course is the reason I'm staying in science as long as I can.

I thank past students including Daniel, Alex, and Stephen.

My committee for the most fun three hours I had each year.

Past and present members of the Gresham and Vogel labs for 
discussions and support.
Especially Niki, Steff, Nathan, and Benjy; Naomi, Farah, Siyu,
Pieter.

To Rodoniki Athansidou for steeling me against the alluring
and corrupting temptation of Kits, and Terrance Cooper for sharing
with me the wisdom of when Kits are necessary for a field to
progress. Threading the gradient between these two extremes seems to
contain some degree of powerful wisdom, something about life and
practice and cargo cults and the very nature of scientific
investigations and Darwinian versus Lamarkian biology, but I have 
not yet had the time to condense it into a sharable form (or a Kit).


the NYU Genomics Core facility for sequencing, flow cytometry,


The aim of sailing is to use lines and fabric and structure to
juxtapose a variety of forces against each other, 
and thus give them the only option of causing progress. 
Without the right constraints and re-direction, 
the system is a mess, destined for the rocks.
A sailboat needs the wind as much as it needs the water,
for without a good keel there's little for the winds to push off of.
Without wind there's little action, and a tiller's important for
deciding where the action should proceed.
Many human endeavors can be described with 
this metaphor --- a variety of forces, pressures, opportunities, 
all swirling around to make something flow in a particular
direction. When all the components align with and against each
other harmoniously, then the effect is made. Just a rope does not make
much progress upwind, and it depends on the cleats and pulleys and
sails and so forth to give it purpose and allow it to play any role in
effecting progress.
 
I would like to acknowledge the system of people that has made 
progress possible.

\fi

\newpage
\pagestyle{plain}

\section*{Abstract}
\addcontentsline{toc}{section}{\numberline{}Abstract}

Cellular responses to changing environments frequently involve rapid
reprogramming of the transcriptome. Regulated
changes in mRNA degradation rates can accelerate reprogramming by
clearing or stabilizing extant transcripts. 
During a nitrogen upshift, the budding yeast \textit{Saccharomyces
cerevisiae} reprograms its physiology in order to enter into  
a state of rapid growth. 
I sought to understand the role of transcriptome reprogramming
in effecting this transition.
During the first minutes of a nitrogen upshift,
preceding the changes in population growth rate,
the transcriptome quickly into a state transiently distinct from
either rapid or slow growth.
During this, the five fastest decreasing mRNA are all NCR-regulated 
transporters, suggesting that this reprogramming is specifically
targeted to set the transcriptional program for rapid growth.
Many transcripts decrease significantly faster than expected from
their stability measured during steady-state growth during
rich media conditions, so I set out to determine the stability of
the yeast transcriptome preceding and during a nitrogen upshift.
I used 4-thiouracil labeling to track mRNA stability, and found 
that 78 mRNAs are subject to destabilization. 
These transcripts include
Nitrogen Catabolite Repression (NCR) and carbon metabolism mRNAs,
suggesting that mRNA destabilization is a mechanism for targeted
reprogramming. 
To explore the molecular basis of destabilization and possible
mechanisms of specifying or effecting the destabilization, I
developed a novel method combining mRNA FISH, fluorescence-activated
cell sorting, and DNA barcode sequencing to screen the pooled deletion
collection library for \textit{trans} factors that mediate rapid GAP1 mRNA
repression. This required that I completely redesign the methods for 
low-input multiplexed barcode sequencing, as well as adapt 
branched-DNA single-molecule mRNA FISH protocols to work in budding
yeast. Modeling of the data identifies known factors of mRNA
degradation, namely all three non-essential components of the
Lsm1-7p/Pat1p complex, as being important for proper \textit{GAP1}
mRNA dynamics. However, the modulators \textit{EDC3} and \textit{SCD6}
have more complex phenotypes. Re-analyzing previously collected data,
I identified that a \textit{scd6}$\Delta$, \textit{tif4632}$\Delta$
(eIF4G2 knock-out), and a \textit{GAP1} 5' UTR delete strain all
share a similar phenotype of lower \textit{GAP1} expression preceding
the upshift, and reduced decay upon the upshift. This suggests that 
translation initiation dynamics may play a role in mRNA levels and 
their stability, and suggests that the destabilization phenomenon 
may in fact be the release of a stabilization effect.

\newpage

{\SingleSpacing
\tableofcontents}

\listoffigures

\mainmatter

\include{dissertation_darachmiller_1}
\include{dissertation_darachmiller_2}
\include{dissertation_darachmiller_3}
\include{dissertation_darachmiller_4}
\include{dissertation_darachmiller_5}

\SingleSpacing

\printbibliography[heading=bibintoc,title={References}]

\end{document}


