\chapter{Conclusion}
\label{chapter:five}

%The nitrogen-upshift in budding yeast is a model system for studying
%how diverse mechanisms of regulation converge on remodeling cellular
%physiology and reprogramming the molecular specifics to 
%carry out the feasting (or yeasting) of the \textit{Saccharomyces
%cerevisiae} feast-and-famine life cycle.

Upon repletion of nitrogen, \textit{Saccharomyces cerevisiae} 
resumes rapid growth.
This involves physiological remodeling as discussed in
\autoref{chapter:four}.
Rapid transcriptional reprogramming also occurs to repress mRNA of 
genes newly unneeded in the replete nitrogen condition, especially
NCR-regulated transporters as seen in \autoref{chapter:two} and
\autoref{chapter:three}.
In efforts to determine the genetic basis of the rapid clearance of
\textit{GAP1} mRNA in particular, I found that decapping modulators
play a role in these dynamics, as discussed in \autoref{chapter:three}. 

\section{Summary}

Here, I summarize these findings and speculate on future
directions that this work could contribute to.

\subsection{mRNA destabilization hastens functional reprogramming}

The five fastest decreasing mRNA in the transcriptome upon the
nitrogen upshift are NCR transporter mRNAs
(\autoref{fig:airoldi2016f6}).
Using principal components analysis, 
we find that the changes in mRNA abundances across the transcriptome
during the upshift are in some ways distinct from simply a switch
from a slow-growing gene expression state to a fast-growing gene
expression state (\autoref{fig:longTermPCA}).
Instead, there is an enrichment in promotion of RNA production and
processing, and a repression of cytoskeleton and membrane
organization (\autoref{subsec:pcaGoCorr}).

mRNA degradation rate control is known to play a role in other
environmental transitions, so I used a 4-thiouracil label-chase
experiment with RNAseq to characterize the stability and changes
of stability of individual mRNA in the yeast transcriptome 
(\autoref{fig:figure2a}).
To avoid problems of nitrogen-regulation of the uracil transporter
necessary for labeling changes, I used an interrupted chase
design.
To smooth technical noise inherent in the normalization of sequencing
data to low amounts of spike-ins, I used a normalization that models
the change of the whole yeast transcriptome.
This prevents particularly noisy measurements of spike-ins from 
being interpreted as changes in mRNA dynamics 
(\autoref{subsec:4tuNormalization}).
To address the potential confounding of synthesis rate changes with
measures of destabilization, I used modeling to characterize the
expected error and threshold our calls of significance on the
effect size (\autoref{subsec:4tuNormalization}).
These methods may assist the future application of the 4tU label-chase
approach to studying extant mRNA stability across dynamic conditions.

Using these methods, I estimated mRNA degradation rates for the
yeast transcriptome and found a median half-life of 6.92 minutes
(\autoref{tab:table1}).
This is less than previous estimates in rich media, suggesting that
mRNA may be less stable during these conditions of limitation of
nitrogen-quality in batch growth conditions. This may reflect 
an evolved preference for flexibility in nitrogen allocation over
the energy costs of turning over mRNA frequently.
Upon the upshift,
I found that degradation rates and changes in
mRNA abundance are, as expected, anti-correlated but not entirely
co-directional in their effect
(\autoref{fig:kkdComparison},\autoref{fig:comparisonDestabilized}).

I found that 78 mRNA are destabilized upon the upshift, with
enrichment for NCR mRNA as well as mRNA encoding factors of carbon
metabolism and vacuole components (\autoref{subsec:destabGoCorr}). 
Amongst these, \textit{GAP1} is subject to an
approximately three-fold increase in degradation rates. 
Thus, mRNA destabilization plays a role in the regulation of this
canonical NCR-regulated mRNA, in addition to the previously known
regulation at layers of mRNA synthesis and post-translational
control.

\subsection{Decapping is important for \textit{GAP1} clearance}

I aimed to determine the genetic factors of this swift clearance, and
so combined mRNA FISH with FACS and sequencing to estimate
\textit{GAP1} mRNA for mutants in a barcoded pool. Development of this
method required several technical advances in optimizing 
\textit{in situ} hybridization methods, developing a robust method 
for barcode sequencing in the face of seemingly inescapable dimer 
formation (\autoref{subsection:bff}), and implementing a modeling
analysis to estimate mutants distributions across bins of 
\textit{GAP1} abundance (\autoref{fig:figure4b}). 

Using this method, I determined that factors of the Lsm1-7p/Pat1p
complex were important for wild-type \textit{GAP1} expression
dynamics (\autoref{fig:figure5a}). Importantly, the modulators
\textit{EDC3} and \textit{SCD6} had defects that did not appear
to generally affect the whole transcriptome, unlike with defects in
\textit{LSM1} and \textit{LSM6} (\autoref{fig:figure5qpcr}).
Fortunately, more analyses of related genes had been completed
by Nathan Brandt, and a re-analysis of this data revealed that 
the Scd6p-interacting eIF4G2/Tif4632p had a similar defect.
Nathan Brandt had also characterized the effects of truncations of
sequence upstream of the \textit{GAP1} start codon or downstream
of the \textit{GAP1} stop codon, and re-analysis showed that the 5'
UTR truncations had a very similar phenotype as deletions of
\textit{SCD6} or \textit{TIF4632}.
These similarities suggest a connection between translation initiation
and mRNA stability during steady-state and dynamic conditions that
may inform our general understanding of the competition of these 
processes.

\subsection{Physiological changes that occur during nitrogen upshift}

Combining a fluorescent poly-dT probe assay
\parencite{amberg1992isolation} with flow cytometry, I was able to
orthogonally confirm the phenomenon of mRNA transcriptome scaling
in different nutrient environments (\autoref{fig:nikis}) and 
measure the dynamics of this transition (\autoref{fig:upshift}).
Given the compatibility of this assay with the SortSeq approach
described in \autoref{chapter:three}, this could be used to efficiently probe the 
genetic factors of transcriptome scaling dynamics in response to
changing environments. Understanding this process in yeast may
inform the study of how c-Myc signalling affects (or effects) 
this process \parencite{nie2012c}.

It is well known that
stress-resistance decreases with an increasing growth rate,
and we demonstrated this is true for a nitrogen-upshift as well
(\autoref{fig:dme180}).
We used this property to enrich a mutant library for defects in
increased susceptibility upon a nitrogen upshift, and found an
enrichment of two components that regulate the cortical actin 
cytoskeleton. Future studies of these, and the two other identified
mutants, may reveal growth-rate regulation of the cortical actin 
cytoskeleton involvement in yeast stress-resistance, and a potential
of Mae1p to connect nitrogen and carbon metabolism.


\section{Suggested future directions}

I would like to expand on these findings to suggest future 
directions of inquiry.

\subsection{An efficient method for estimating mRNA abundance in
barcoded mutant pools}

Much of what we know about mRNA degradation has relied on genetics,
but perturbing an exquisitely homeostatic system, like a living cell, 
can result in indirect effects as systems of regulatory feedback 
percolates the signal through the network. 
Thus in this work, the use of knockout mutants is the use of
an indeterminately perturbed system. 
Additionally, the laborious individual creation and maintenance of a 
collection of mutants can introduce potential artifacts of suppressor 
and passenger mutations \parencite{kwan2016rdna,markowitz2017reduced}.
The development of new methods for making mutant pools with 
high internal replication and minimal strain-handling selection
offers a new paradigm of yeast genetics that avoids or
minimizes some of these drawbacks \parencite{smith2016quantitative}.
The methods described here for doing SortSeq can be adapted
to any DNA-barcode-based sequencing assay of pooled mutants. 

With the integration of single-cell RNA sequencing with 
RNA-guided Cas9 genetic-editing 
\parencite{dixit2016perturb,hill2018design},
this approach of mRNA SortSeq may no longer be absolutely
necessary for doing high-throughput genetics for mRNA markers. 
However, this method works now with common lab reagents and equipment, 
and only requires quantifying the mutants across bins---
not quantifying all mRNA for all mutants.
For questions regarding a small number of transcripts,
the depth of replicates and timepoints possible with a SortSeq
design may be advantageous.
Additionally, the recalcitrance of budding yeast for 
single-cell molecular analysis is still technically challenging
\parencite{gasch2017single}.
Scaling this BFF assay up to more cells input and a greater investment
in FACS sorting \parencite{de2017deciphering} has the potential to 
estimate mRNA dynamics across multiple
timepoints for all barcoded mutants in a library.
However, single-cell RNA sequencing is advantaged in its global 
perspective and will presumably scale with methodological 
developments to increase cell throughput 
(thus replicating measures per genotype)
and sequencing throughput.
The efficiency of the pooled SortSeq approach could also be used to 
limit the search space of more precise but resource-intensive 
automated assays \parencite{worley2016genome}, 
and thus a hybrid approach could be used to
efficiently increase throughput of measured transcript dynamics. 
SortSeq for estimating mRNA abundance, or other steps of gene
expression, may be a useful orthogonal tool complementing
other approaches with its potential for scaling measurements across
timepoints, conditions, variants, and replicates.

\subsection{All metabolism is connected}

We often reduce phenomena to simple models in order to
create a more powerful generality, but sometimes a
powerful categorization can be too far
\parencite{lazebnik2002can}.
In the label-chase experiment, I found that mRNA encoding factors of
carbon metabolism (namely pyruvate metabolism, and the isoenzymes 
\textit{PYK2} and \textit{HXK1}) were destabilized.
In the genetic screen (BFF), we found that mutants in negative
regulation of gluconeogenesis were enriched in mutants with high
\textit{GAP1} mRNA estimated after the shift
(\autoref{fig:gluco}), suggesting defects 
in repression. The core of nitrogen metabolism (glutamate and
glutamine) are intimately connected with the citric acid cycle via
alpha-ketoglutarate, and the roles of these metabolites or their
rates of inter-conversion have been proposed to play roles in
signalling changes in growth \parencite{fayyad2016yeast}.
Thus, continued study of the metabolic networks during perturbations
(like a glutamine upshift) may reveal an unappreciated cross-talk
amongst sub-networks that are currently considered as distinct. 
This may reveal contingent or adaptive regulation of diverse
signalling pathways from secondary metabolite read-outs that are
distal from the experimenter's intended input.
Here, glutamine is considered as a nitrogen-source but delivers to 
the cell two nitrogen atoms and a carbon skeleton that can be
delivered into the citric-acid cycle.
Thus, a systematic view of metabolism during this transitions may
reveal that the effects of a glutamine upshift may be more broad than
just nitrogen metabolism and regulation, and may inform our more
specific interpretation of these perturbation experiments.

\subsection{pH as a possible cause of mRNA destabilization}

Addition of a preferred nitrogen source triggers a destabilization 
for several mRNA. 
A similar phenomenon of mRNA destabilization is observed during the 
re-addition of glucose to cultures limited for growth by carbon 
availability \parencite{braun2016snf1}. 
The mechanisms by which this occurs appear to be diverse, although
they tend to occur via the canonical 5'-decapping mechanism.
How might these two transitions be similar?

Yeast acidify their media via the action of the ATP-dependent
proton export pump \textit{PMA1}, regulating their cytoplasmic pH
at approximately pH 7 (or $\sim$3000 protons in total 
\cite{orij2011intracellular}).
The General Amino-Acid Permease 1 (\textit{GAP1}) and other
transporters use this proton-gradient to symport amino-acids into
the cell \parencite{grenson1992amino}. 
The re-addition of amino acids is expected to depolarize this 
gradient, and \textit{GAP1} mutants who are defective in the 
inactivation of Gap1p upon refeeding have a growth defect
\parencite{risinger2006activity}, suggesting that the process of
active transport is what contributes to the defect in cellular 
metabolism and growth.
Glucose upshifts result in a rapid but transient reduction in pH
\parencite{kresnowati2008quantitative}, thought to result from
a sudden increase in proton-release during the early steps of 
glycolysis \parencite{orij2011intracellular}.
Similarly, the catabolism of glutamine into alpha-ketoglutarate
and free ammonium ions should produce one proton per glutamine
\parencite{magasanik2002nitrogen}, in addition to the proton
from symport of one glutamine.

Both of these upshifts are likely to decrease cytoplasmic pH.
This could affect mRNA stability via effects through signalling
pathways like Ras/PKA, which is known to affect the signalling
of this pathway through cAMP \parencite{orij2011intracellular}.
Recently it has been shown that the proton-influx from nutrient uptake
via symport mechanisms is implicated in directly causing the spike
in TORC1 activity normally attributed to the nutrient source 
\parencite{saliba2018yeast}. 
The GAAC pathway via the Gcn2p kinase has been shown to respond
and be important for responding to acidification 
\parencite{hueso2012novel}.
Single or combinations of these pathways may signal to alter
mRNA degradation rates.

Alternatively, changing pH can have strong biophysical effects on
protein-protein interactions and folding that may perturb mRNA
degradation metabolism. 
Pab1 has been recently shown to have a pH-dependent tendency to
condense in response to stressors, a process that affects its 
function likely through Ccr4-dependent deadenylation
\parencite{riback2017stress,yao2007pab1}.
Additionally, the condensation of mRNA processing-bodies is 
correlated with impacts on the pace of mRNA degradation metabolism 
in the yeast cytoplasm \parencite{huch2017mrna, rao2017numerous}.
Thus large changes in the pH may directly perturb the molecular
interactions that effect the processes of mRNA degradation.

pH changes as directly impacting the molecular processes that are
sensitive to changes in post-translational modifications and
localization would be a mechanism to explain the general phenomenon
of mRNA stability changes during environmental changes
\parencite{perez2013eukaryotic, canadell2015impact}.
The role of metabolism or proton-symporters in causing this would
explain some of the generality of the phenomenon that has been
described as "transceptors" in the work of Theiveilen.

Future work to define the extent and impact of this phenomenon
would make a significant impact on our understanding of cytoplasmic
processes and the determinants of mRNA degradation metabolism. 
Precisely defining the changes in cytoplasmic pH
on the seconds timescale during these environmental changes would 
define the timescales of interest. Metabolic label-chase experiments
with mutants allow a genome-wide assay of functional impact.
Live-cell microscopy in micro (or milli) fludics devices would
allow the integration of these datasets to test if
molecular-condensation events may play a role, a hypothesis
that could be explored mechanistically with biochemical 
\textit{in vitro} work.

\subsection{Possible mechanisms of \textit{GAP1} clearance}

\textit{GAP1} mRNA is now an example of mRNA destabilized upon
a nutrient upshift. What effects this regulation?
I identified several candidate factors, here I speculate on the
implications of one group of factors.

Scd6p interaction with eIF4G1 has been primarily studied, 
but here we see a similar phenotype with eIF4G2 
(eIF4G1 was not tested). These are
homologs with very similar functionality
\parencite{clarkson2010functional}, 
although eIF4G1 is expressed higher
during “normal” laboratory growth conditions, ie rich media. 
eIF4G2 has been shown to be more highly expressed and localized to
processing-bodies during conditions of nutrient limitations, and
this is suggested to specify the storage of mRNA in anticipation of
resuming rapid growth \parencite{brengues2007accumulation}.
This initiation factor may help to globally (or specifically)
reduce the initiation frequency, thus reducing the fraction of
ribosomes consuming tRNA under conditions of low amino-acid,
and thus low charged-tRNA, availability (\autoref{fig:trna})
This would help ensure an adequate
supply of charged-tRNAs for elongation of engaged ribosomes.
Differential regulation of 
translation initiation could explain the observation of sub-maximal
usage of extant ribosomes in initiation-limited regimes of cellular
growth \parencite{kafri2016cost,metzl2017principles}.
Continued studies of the role of Scd6p in specifying ribosome
loading in diverse environmental conditions could reveal a mechanism
to effect these adaptive changes in translation.
Additionally, a systematic study of the \textit{GAP1} 5' UTR could
reveal \textit{cis}-elements necessary for this regulation.
In particular, the study of the rapid changes that occur during the
nitrogen upshift may shed light on their role during steady-state
growth.

\vspace{2em}

\afig{
  \begin{tikzpicture}[scale=0.9
    ,every node/.style={scale=0.9}
    ,>={Latex[length=3mm,width=3mm]}
%    ,snakey/.style={decorate,decoration={snake,segment length=0.75cm}}
    ,textz/.style={ font=\scriptsize,align=left }
    ]

\foreach \y/\riboOne/\riboTwo/\riboThree in {0.0/-1/-1/-1,-3/0/2/4,-6/0/0.75/1.5,-9/3/3/3} {
  % First translating one
    \draw[thick,color=red] (-2.0,\y) -- (5,\y);
    \node[anchor=east,font=\tiny] at (-2.0,\y) {m7G};
    \node[anchor=west,font=\tiny] at (5,\y) {AAAAAAAAAAAAAAAAA};
    \newcommand*{\pablif}{\y+0.5}
    \node[circle,fill=yellow!80!black,textz] at (6.5,\pablif) {Pab1p};
    \node[anchor=east,font=\tiny,circle,fill=yellow] at
      (-2.8,\y) (eif4f1) {eIF4F};
    \newcommand*{\ribodrop}{\y-0.1}
    \newcommand*{\ribolif}{\y+0.1}
    \draw[fill=blue,color=blue!50] (\riboOne,\ribodrop) ellipse (0.16cm and 0.08cm);
    \draw[fill=blue,color=blue!50] (\riboOne,\ribolif) ellipse (0.16cm and 0.16cm);
    \draw[fill=blue,color=blue!50] (\riboTwo,\ribodrop) ellipse (0.16cm and 0.08cm);
    \draw[fill=blue,color=blue!50] (\riboTwo,\ribolif) ellipse (0.16cm and 0.16cm);
    \draw[fill=blue,color=blue!50] (\riboThree,\ribodrop) ellipse (0.16cm and 0.08cm);
    \draw[fill=blue,color=blue!50] (\riboThree,\ribolif) ellipse (0.16cm and 0.16cm);
}

  \node[align=left,font=\small,anchor=west] at (-6,0.0) 
    {Nitrogen\\rich, high \\ initiation};
  \draw[->,very thick] (-2.8,0) to[out=45,in=135] node[above,pos=0.5,textz]
    {loads ribosome} (-1.2,0.2);
  \draw[->,thick,color=green] (-0.6,-0.5) to (-0.0,-0.5);
  \draw[->,thick,color=green] (-1.3,-0.5) to (-0.7,-0.5);
  \draw[->,thick,color=green] (-2.0,-0.5) to (-1.4,-0.5);
  \node[align=left,font=\small,anchor=west] at (1,-1.0) 
    {abundant charged tRNAs, fast elongation};

  \node[align=left,font=\small,anchor=west] at (-6,-3.0) 
    {Nitrogen\\rich, high \\ initiation};
  \draw[->,very thick] (-2.8,-3) to[out=45,in=135] (-1.2,-2.8);
  \draw[->,thick,color=green] (-1.0,-3.5) to (-0.4,-3.5);
  \draw[->,thick,color=green] (-0.3,-3.5) to (0.3,-3.5);
  \draw[->,thick,color=green] (0.4,-3.5) to (1.0,-3.5);
%
  \draw[->,thick,color=green] (1.1,-3.5) to (1.6,-3.5);
  \draw[->,thick,color=green] (1.7,-3.5) to (2.3,-3.5);
  \draw[->,thick,color=green] (2.4,-3.5) to (2.9,-3.5);
%
  \draw[->,thick,color=green] (3.0,-3.5) to (3.6,-3.5);
  \draw[->,thick,color=green] (3.7,-3.5) to (4.3,-3.5);
  \draw[->,thick,color=green] (4.4,-3.5) to (5.0,-3.5);

  \node[align=left,font=\small,anchor=west] at (-6,-6.0) 
    {Nitrogen\\poor, high \\ initiation};
  \draw[->,very thick] (-2.8,-6) to[out=45,in=135] (-1.2,-5.8);
  \draw[->,thin,color=green] (-0.3,-6.5) to (0.3,-6.5);
  \draw[-|,red,color=red] (0.4,-6.5) to (1.0,-6.5);
  \draw[-,color=gray] (1.1,-6.5) to (1.7,-6.5);
  \node[align=left,font=\small,anchor=west] at (1,-7.3) 
    {scarce charged tRNAs per ribosome, \\ 
      inconsistent elongation, ribosome stalling \\ and collisions};

  \node[align=left,font=\small,anchor=west] at (-6,-9.0) 
    {Nitrogen\\poor, low \\ initiation};
  \draw[->,dashed] (-2.8,-9) to[out=45,in=135] (-1.2,-8.8);
%  \draw[->,thick,color=yellow] (-0.3,-9.5) to (0.3,-9.5);
  \draw[->,thick,color=yellow] (2.7,-9.5) to (3.3,-9.5);
  \node[align=left,font=\small,anchor=west] at (1,-10.3) 
    {match elongating ribosomes to charged tRNAs , \\ 
      more consistent elongation, less stalling \\ and collisions};

  \node[align=left,font=\small,anchor=west] at (9,-6) 
    {\textit{GAP1} mRNA, \\ mutant \\ 5' UTR or \\ \textit{scd6}$\Delta$};

  \node[align=left,font=\small,anchor=west] at (9,-8.5) 
    {wild-type \\ \textit{GAP1} mRNA};

  \end{tikzpicture}
  }{
    Low tRNA charging could limit translation elongation rates,
    and this could result in ribosome collisions that have been
    proposed to trigger quality control mechanisms. 
    Limiting translation initiation in an elongation-limited regime 
    would prevent such collisions and conserve the tRNA pool to
    allow engaged ribosomes to finish elongation.
    \label{fig:trna}
  }
  {Hypothetical adaptive role for slowed translation under low
   amino-acid conditions}

The translation and degradation of mRNA are intimately coupled
processes which compete for the same substrate, the mRNA. 
Very recently, imaging studies have challenged the 5'-3' closed loop
model of mRNA \parencite{adivarahan2017spatial}, and demonstrated
a strong effect of the translational status of ribosomal loading in 
separating the 5' and 3' ends of a single mRNA.
Degradation is mediated by connections between 3' and 5' ends, 
with the Lsm1-7p/Pat1p complex strongly promoting the decapping of
mRNA requisite for degradation in the main pathway of mRNA
degradation.
One explanation of the interaction between the degradation and
translation initiation machinery has been proposed as the competition
for the 5' m7G cap. 
The interaction of translation initiation factor eIFG2 and
decapping modulator Scd6p in affecting the degradation of
\textit{GAP1} suggests and interaction with ribosome loading status.
One explanation that connects these processes could be that the presence
of elongating ribosomes separates the 5' and 3' ends of the mRNA, 
similar to the cohesin-dependent loop extrusion model currently used 
by some to explain chromatin-organization (\autoref{fig:riboextrusion}).
This would directly explain the connection between translation status
and degradation for individual mRNA as a simple function of 
3'-associated decapping factors stochastically interacting with 
the 5' end of the mRNA tether.

\afig{
  \begin{tikzpicture}[scale=0.9
    ,every node/.style={scale=0.9}
    ,>={Latex[length=2mm,width=2mm]}
    ,textz/.style={ font=\scriptsize,align=left }
    ]

  \node[align=left,font=\small,anchor=west] at (-4,1) 
    {wild-type \textit{GAP1} mRNA, actively translating during nitrogen-limitation\\
      separated 5' and 3' subcomplexes rarely interact, low degradation probability};
  \draw[fill=red!80!black,color=red!80!black] (9.0,-1.0) arc (0:360:1) --
    (8.5,-1) arc (0:-360:0.5) -- cycle;
  \draw[decorate,decoration={text along path,text=|\footnotesize|Lsm1-7p/Pat1p complex}]
    (6.8,-1) arc (180:0:1.2);
  \draw[thick,color=red] (-2.0,-1.0) 
    .. controls (-1,0.0) and (0,-2) .. (1,-1)
    .. controls (2,0.0) and (3,-2) .. (4,-1)
    .. controls (5,0.0) and (6,-2) .. (7,-1);
  \node[anchor=east,font=\tiny] at (-2.0,-1.0) {m7G};
  \node[anchor=west,font=\tiny] at (7,-1.0) {AAAAAA};
%{Lsm1-7p/Pat1p\\complex};
  \node[anchor=east,font=\tiny,circle,fill=yellow] at
    (-2.8,-1.0) (eif4f1) {eIF4F};
  \draw[fill=blue,color=blue!50] (-0.5,-1.0+0.2) ellipse (0.32cm and 0.16cm);
  \draw[fill=blue,color=blue!50] (-0.5,-1.0-0.2) ellipse (0.32cm and 0.32cm);
  \draw[fill=blue,color=blue!50] (1.6,-.8+0.2) ellipse (0.32cm and 0.16cm);
  \draw[fill=blue,color=blue!50] (1.6,-.8-0.2) ellipse (0.32cm and 0.32cm);
  \draw[fill=blue,color=blue!50] (5.0,-.8+0.2) ellipse (0.32cm and 0.16cm);
  \draw[fill=blue,color=blue!50] (5.0,-.8-0.2) ellipse (0.32cm and 0.32cm);

%

  \node[align=left,font=\small,anchor=west] at (-4,-4) 
    {wild-type \textit{GAP1} mRNA, halt in translation due to N-upshift \\
      5' and 3' subcomplexes are tethered, this passively promotes interaction,\\
      high degradation probability};

  \draw[fill=red!80!black,color=red!80!black] (6.0,-6.0) arc (0:360:1) --
    (5.5,-6) arc (0:-360:0.5) -- cycle;
  \draw[decorate,decoration={text along path,text=|\footnotesize|Lsm1-7p/Pat1p complex}]
    (3.8,-6) arc (180:0:1.2);
  \draw[thick,color=red] (-1.0,-6) 
    .. controls (0,-10) .. (0.5,-8)
    .. controls (1.0,-3) and (1.5,-11) .. (2.0,-8)
    .. controls (2.5,-3) and (3.0,-11) .. (3.5,-8)
    .. controls (3.75,-6) .. (4,-6) ;
  \node[anchor=east,font=\tiny] at (-1.0,-6.0) {m7G};
  \node[anchor=west,font=\tiny] at (4,-6.0) {AAAAAA};
%{Lsm1-7p/Pat1p\\complex};
  \node[anchor=east,font=\tiny,circle,fill=yellow] at
    (-1.8,-6.0) (eif4f1) {eIF4F};


  \draw[fill=red!80!black,color=red!80!black] (4.0,-13.0) arc (0:360:1) --
    (3.5,-13) arc (0:-360:0.5) -- cycle;
  \draw[decorate,decoration={text along path,text=|\footnotesize|Lsm1-7p/Pat1p complex}]
    (1.8,-13) arc (180:0:1.2);
  \draw[thick,color=red] (-1,-13) 
    .. controls (-2.5,-17) and (-1.5,-16) .. (-1.0,-15)
    .. controls (-0.5,-14) and (0.0,-17) .. (0.5,-15)
    .. controls (1.0,-14) and (1.5,-17) .. (2.0,-13);
  \node[anchor=south,font=\tiny] at (-1.0,-13) {m7G};
  \node[anchor=west,font=\tiny] at (2,-13) {AAAAAA};
%{Lsm1-7p/Pat1p\\complex};
  \node[anchor=south,font=\tiny,circle,fill=yellow] at
    (-1.0,-12.6) (eif4f1) {eIF4F};

  \draw[->,very thick,color=red] (1.5,-12) to[bend right] (-0.5,-12.5);
  \node[anchor=south,font=\small] at (0.5,-11.5) {decapping};

  \end{tikzpicture}
  }{
    In the crowded cytoplasm, mRNA are not likely to resemble 
    straight lines. The tethering of the 5' to the 3' end by the
    mRNA molecule would be expected to drive interaction of the two
    RNP subcomplexes. Some evidence points toward ribosomal 
    elongation as a process by which the two halves are separated,
    and would be a simple explanation for the observation that
    inhibiting either elongation or initiation appears to affect
    mRNA degradation rates, given that the assembly of the decapping
    complex with the assistance of the 3' Lsm1-7p/Pat1p complex 
    appears to important for decapping.
    \label{fig:riboextrusion}
  }
  {Hypothetical mechanism by which translation initiation and 
   elongation inhibit mRNA degradation --- ribosome extrusion.}




