\chapter{Measuring the extent of mRNA destabilization and screening 
for genetic factors of \textit{GAP1} repression}
\label{chapter:three}

This chapter is similar to an article currently submitted to a 
journal for review and publication. 
It is also posted on \textit{biorxiv}, titled:
\textit{"Global analysis of gene expression dynamics identifies factors
required for accelerated mRNA degradation"}.
Authorship of this article is: \textbf{Darach Miller}, Nathan Brandt, 
and David Gresham.
Darach Miller did most of the benchwork, analysis, and writing.
Nathan Brandt did all benchwork to generate mutants and collect qPCR 
for \autoref{fig:figure5qpcr}.
David Gresham helped with discussions and co-writing the article.
The \textit{biorxiv} version is at \url{doi.org/10.1101/254920}.

The below is adapted for the dissertation, incorporating important
text from the supplementary methods into the chapter text.
Supplemental tables are available from the OSF repository linked 
with this work (\url{https://osf.io/7ybsh/files/}), 
and are reproducible
using the \texttt{Makefile} and associated scripts
in the git repository distributed with the paper
(\url{http://github.com/darachm/millerBrandtGresham2018}).

%The methods here allow
%for the use of fixed-cell flow-cytometry assays in pooled Sort-seq
%assays on yeast, and would be useful to inform the development of
%similar assays in other systems. Development of this approach to
%estimating mRNA abundance on pooled mutants would enable the
%combination of transcriptomics as a high-dimensional marker of
%cellular signalling pathways with the use of transcript markers to
%explore the genetics of these pathways.  Supplementary issues
%Pulse-chase modeling ( I basically want to reprint the stuff in the
%supplement here ) BFF rationale, methodology, and future directions (
%I basically want to reprint the stuff in the supplement here, then
%waste paper speculating )

\section{Abstract}

Cellular responses to changing environments frequently
involve rapid reprogramming of the transcriptome.
Regulated changes in mRNA degradation rates can
accelerate reprogramming by clearing or stabilizing extant transcripts. 
%Budding yeast respond to an improvement in
%nitrogen-availability by triggering a transcriptional reprogramming
%that functions to upregulate ribosome biogenesis and repress
%alternative nitrogen-source catabolism. 
Here, we measured mRNA stability using 4-thiouracil (4tU) labeling
in the budding yeast \textit{Saccharomyces cerevisiae}
during a nitrogen upshift and found that 78 mRNAs are subject
to significant destabilization. These transcripts include 
genes involved in Nitrogen
Catabolite Repression (NCR) and carbon metabolism mRNAs,
suggesting that mRNA destabilization is a mechanism 
for targeted reprogramming.
To explore the molecular basis of
destabilization, we implemented a SortSeq approach 
using a pooled deletion collection library
to screen for \textit{trans} factors that mediate rapid \textit{GAP1}
mRNA repression.
We combined low-input multiplexed \underline{B}arcode sequencing 
with branched-DNA single-molecule mRNA \underline{F}ISH and 
\underline{F}luorescence-activated cell sorting (\underline{BFF})
to identify that the Lsm1-7p/Pat1p complex and general mRNA
decay machinery are important for \textit{GAP1} mRNA clearance.
We also find that the decapping modulator \textit{SCD6}, translation
factor eIF4G2, and the 5' UTR of \textit{GAP1}
are important for this repression, 
suggesting that translational control may impact the 
post-transcriptional fate of mRNAs in response to 
environmental changes.

\section{Introduction}

Regulated changes in mRNA abundance are a primary cellular response
to external stimuli.
Both the rate of synthesis and the rate of degradation determine the
steady-state abundance of a particular mRNA and the kinetics
with which abundance changes occur
\parencite{hargrove1989role,perez2013eukaryotic}. 
Changes in mRNA degradation rates fulfill an important 
mechanistic role in diverse systems, including 
development \parencite{alonso2012complex,west2018developmental} and disease
\parencite{aghib19903}.
In budding yeast, the rate of
mRNA degradation is affected by environmental stresses
\parencite{canadell2015impact}, cellular growth rate
\parencite{garcia2016growth}, and by improvements in 
nutrient conditions \parencite{scheffler1998control}.

Environmental shifts trigger rapid reprogramming of the budding yeast
transcriptome in response to stresses and nutritional
changes \parencite{gasch2000genomic,conway2012glucose}. mRNA degradation rate changes
have been shown to play a role in responses to heat-shock, osmotic
stress, pH increases, and oxidative stress
\parencite{castells2011heat,romero2009specific,canadell2015impact,molina2008comprehensive}. 
In response to these diverse stresses, 
destabilization of mRNAs encoding 
ribosomal-biogenesis gene products and  
stress-induced mRNA occurs \parencite{canadell2015impact}. 
Simultaneous increases in both synthesis and
degradation rates of some  mRNAs may serve to speed the return to a
steady-state following a transient pulse of regulation
\parencite{shalem2008transient}. Addition of glucose to carbon-limited cells 
results in both stabilization of 
ribosomal protein mRNAs \parencite{yin2003glucose} and destabilization
of gluconeogenic transcripts \parencite{de2002role,mercado1994levels}.
Destabilization of transcripts can
have a delayed effect on reducing protein levels compared to
up-regulated genes \parencite{lee2011dynamic}. This suggests that accelerated
mRNA degradation may serve additional purposes. For example, clearance
of specific mRNAs could increase nucleotide pools
\parencite{kresnowati2006transcriptome} or facilitate reallocation of
translational capacity 
\parencite{kief1981coordinate,giordano2016dynamical,shachrai2010cost}.
%Identifying the genetic
%factors responsible for the accelerated mRNA degradation would allow
%us to test if regulated destabilization of specific transcripts is
%adaptive.

Yeast cells metabolize a wide variety of nitrogen sources, but
preferentially assimilate and metabolize specific nitrogen compounds.
Transcriptional regulation, known as
“nitrogen catabolite repression” (NCR)
\parencite{magasanik2002nitrogen},
controls the expression of mRNAs
encoding transporters, metabolic enzymes, and regulatory
factors required for utilization of alternative nitrogen sources. 
NCR-regulated transcripts are expressed in the
absence of a readily metabolized (preferred) nitrogen sources or in
the presence of growth-limiting concentrations (in the low $\mu$M range)
of any nitrogen source \parencite{godard2007effect,airoldi2016steady}. Regulation
of NCR targets is mediated by two activating GATA
transcription factors, Gln3p and Gat1p, and two repressing
GATA factors, Dal80p and Gzf3p. \textit{GAT1}, \textit{GZF3}, and
\textit{DAL80} promoters
contain GATAA motifs, and thus transcriptional regulation of NCR
targets entails self-regulatory and cross-regulatory loops. When
supplied with a preferred nitrogen source such as glutamine, the
NCR-activating transcription factors Gat1p and Gln3p are excluded from
the nucleus by TORC1-dependent and -independent mechanisms
\parencite{beck1999tor,tate2013five,tate2017general} and NCR transcripts are strongly
repressed. The activity of some NCR gene
products are also controlled by post-translational mechanisms
\parencite{cooper1983function} such as the General Amino-acid Permease
(Gap1p) which is rapidly inactivated upon a nitrogen 
upshift via ubiquitination
\parencite{stanbrough1995transcriptional,risinger2006activity,merhi2012internal}. Recently, we have
identified an additional level of regulation of NCR transcripts: cells
growing in NCR de-repressing conditions accelerate the degradation
of \textit{GAP1} %and \textit{DIP5} mRNAs
mRNA upon addition of glutamine
\parencite{airoldi2016steady}. Thus, mRNA degradation rate regulation may be an
additional mechanism for clearing NCR-regulated transcripts upon 
improvements in environmental nitrogen availability.

Multiple pathways mediate the degradation of mRNAs. The main pathway
of mRNA degradation occurs by deadenylation and decapping
prior to 5' to 3' exonucleolytic degradation by Xrn1p; however,
transcripts are also degraded 3' to 5' via the exosome, or via
activation of co-translational quality control mechanisms
\parencite{parker2012rna}. Deadenylation of mRNAs by the Ccr4-Not complex
allows the mRNA to be bound at the 3' end by the 
Lsm1-7p/Pat1p complex, a heptameric
ring comprising the SM-like proteins Lsm2-7p and the
cytoplasmic-specific Lsm1p
\parencite{tharun2000yeast,sharif2013architecture}, which then
recruits factors for decapping by Dcp2p. 
Recruitment of the decapping enzyme \parencite{coller2004eukaryotic} is the 
rate-limiting step for canonical 5'-3' degradation.
Therefore Lsm1-7p, Pat1p,
and associated factors play a key role \parencite{nissan2010decapping}. 

Regulation of mRNA degradation pathways can alter the stability of
specific mRNAs. For example, the RNA-binding protein (RBP) Puf3p
recognizes a \textit{cis}-element in 3' UTRs \parencite{olivas2000puf3}
and affects mRNA degradation rates depending on
Puf3p phosphorylation status \parencite{lee2015glucose}. 
%Transcript properties
%also associated with translation dynamics affect mRNA degradation, at
%the level of elongation \parencite{Sweet2012,Presnyak2015,Neymotin2016} or
%competition between the decapping enzymes and translation initiation
%\parencite{Schwartz2000}. 
In addition to \textit{cis}-elements within the transcirpt, 
promoters have
been shown to mark certain RNA-protein (RNP) complexes to specify
their post-transcriptional regulation
\parencite{mercado1994levels,haimovich2013gene,trcek2011single,braun2016snf1}. These
mechanisms may be controlled by a variety of different signalling
pathways including Snf1
\parencite{young2012amp,braun2014phosphoproteomic}, PKA
\parencite{ramachandran2011camp}, Phk1/2 \parencite{luo2011nutrients}, and TORC1
\parencite{talarek2010initiation}. Thus, regulated changes in  mRNA degradation
rates includes numerous mechanisms that collectively tune stability of
mRNAs in response to the activity of signalling pathways. 

Here, we studied the global regulation of mRNA degradation rates upon
improvment in environmental nitrogen using 4-thiouracil (4tU) 
label-chase and RNAseq.
We found that a set of 78 mRNAs are subject to accelerated mRNA
degradation, including many NCR transcripts as well as mRNAs
encoding components of
carbon metabolism. To identify the mechanism underlying accelerated
mRNA degradation we designed a high-throughput genetic screen using 
\underline{B}arcode-sequencing of a pooled library which was
fractionated using \underline{F}luorescence-activated cell 
sorting of single molecule mRNA \underline{F}ISH signal (BFF). 
We screened the barcoded
yeast deletion collection to test the effect of each gene deletion
on the abundance of \textit{GAP1} mRNA in NCR de-repressing 
conditions and its clearance following the 
addition of glutamine. We
find that the Lsm1-7p/Pat1p complex and decapping modifiers affect
both \textit{GAP1} mRNA steady-state mRNA abundance and its 
accelerated degradation.
This work expands our
understanding of mRNA stability regulation in remodeling the
transcriptome during a relief from growth-limitation and demonstrates
a generalizable approach to the study of genetic determinants of mRNA
dynamics.


%%%%%
%%%%%
%%%%%
%%%%%
%%%%%

\section{Characterizing transcriptome dynamics upon a nitrogen upshift}

\subsection{Transcriptional reprogramming precedes physiological remodeling}

Cellular responses to environmental signals entail coordinated changes
in both gene expression and cellular physiology.  Previously, we
studied the steady-state and dynamic responses of 
\textit{Saccharomyces cerevisiae} 
(budding yeast) to environmental nitrogen
\parencite{airoldi2016steady}, and found that the transcriptome is rapidly
reprogrammed following a single pulsed addition of glutamine to
nitrogen-limited cells in either a chemostat or
batch culture. To study physiological changes in response to a
nitrogen upshift, we measured growth rates of a population of 
cells. A prototrophic haploid lab strain 
(FY4, isogenic to S288c) grows with a
4.5 hour doubling time in batch culture in minimal media 
containing proline as a sole
nitrogen source (\autoref{fig:figure1a}). Upon addition of 400$\mu$M glutamine
the cells undergo a 2-hour lag period during which no change in
population growth rate is detected, but the average cell size
continuously increases ($\sim$21\% increase in mean volume 
\autoref{fig:figure1b}). Following the lag, the population adopts a 2.1 
hour doubling time.
%This lag in population growth rate upon an upshift has been 
%described before \parencite{Carter1978}.
By contrast, global gene expression changes are detected
within three minutes of the upshift \parencite{airoldi2016steady}. 
Thus, transcriptome remodeling precedes
physiological remodeling in response to a nitrogen upshift.

\afig{
  \includegraphics[width=.8\textwidth]{img/Figure1a.png}
  \includegraphics[width=.8\textwidth]{img/Figure1b.png}
  }{
    \textbf{(Top)} 
    400$\mu$M glutamine was added to a
    culture of yeast cells growing in minimal media containing 800$\mu$M
    proline as a sole nitrogen source. Measurements
    of culture density across the upshift are plotted. 
    Dotted lines denote linear regression of the
    natural log of cell density against time before the upshift and 
    after the 2 hour lag. \textbf{(Bottom)} Average cell size.
    Dotted lines denote the mean cell diameter before the upshift
    and after the 2 hour lag. 
    \label{fig:figure1a}
    \label{fig:figure1b}
  }{The nitrogen upshift of population and cellular growth rate.}

\afig{
  \includegraphics[width=\textwidth]{img/Figure1c.png}
  }{
    PCA analysis of global
    mRNA abundance in steady-state chemostats and following an upshift
    \parencite{airoldi2016steady}. Steady-state nitrogen-limited chemostat
    cultures maintained at different growth rates (colored circles)
    primarily vary along principal component 2. mRNA abundance changes
    following a
    nitrogen-upshift in either a chemostat (squares) or batch culture
    (triangles) show similar trajectories and primarily vary along
    principal component 1. Grey lines depict the major trajectory
    of variation for the steady-state and upshift experiments.
    \label{fig:figure1c}
  }{Dynamics of transcriptome remodeling during a nitrogen upshift, 
    on a coarse scale.}

\afig{
  \includegraphics[width=\textwidth]{img/Figure1_S_longTermPCA.png}
  }{
  Principal components analysis (SVD) of microarray data from 
  \cite{airoldi2016steady}. 
  Colored points are from steady-state chemostats grown in
  nitrogen-limitation with various nitrogen sources, 
  at different growth rates.
  Time-series experiments are show in grey points, connected by lines,
  and line-type is the type of upshift (in batch or in chemostat).
  \label{fig:longTermPCA}
  }{The coarse long-term transcriptome dynamics of a glutamine upshift.}

To evaluate concordance in transcriptome remodeling between chemostat
and batch nitrogen upshifts, and the extent to which they reflect
changes in mRNA abundance observed during systematic steady-state 
changes in growth rates using chemostats, we
performed principal component analysis on microarray data
(\autoref{fig:figure1c}). The first two principal components, which
account for almost half of the total variation, clearly separate
steady-state and nitrogen upshift cultures.  Systematic changes in
growth rate primarily results in
a separation of states along the second principal
component, whereas upshift experiments vary along the first 
principal component.  This suggests that the
transcriptome is remodeled through a distinct state. 
In upshift experiments in
chemostats, the trajectory ultimately returns to 
the initial steady-state condition as excess nitrogen is 
depleted by consumption and dilution 
(\autoref{fig:longTermPCA}). 


To investigate the functional basis of mRNA abundance changes
in the upshift and steady-state conditions, we computed the
correlation of each transcript with the loadings on these first two
principal components and performed gene-set enrichment analysis
(\nameref{itm:microarrayPCAgsea}). 
Component 1 is positively correlated with functions like
mRNA processing, transcription from RNA polymerases (I,II,and III),
and chromatin organization, and negatively correlated with
cytoskeleton organization,
vesicle organization, membrane fusion, and cellular respiration.
Both steady-state and upshift 
trajectories increase with principal component 2, but they diverge
along principal component 1. Components 1 and 2 are 
strongly enriched for terms including ribosome biogenesis, 
nucleolus, and
tRNA processing, and negatively correlated with
vacuole, cell cortex, and carbohydrate metabolism terms. 
Together, this analysis suggests that both upshift and
increased steady-state growth rates share upregulation of
ribosome-associated components, but the reprogramming
preceding the upshift in growth reflects an immediate focus on 
gene expression machinery instead of structural cellular components.
Importantly,
dynamic reprogramming is similar in both the chemostat and batch
upshift (\autoref{fig:figure1c}). As batch cultures are a technically
simpler experimental system, we performed all subsequent experiments
using batch culture nitrogen upshifts. 

\label{subsec:pcaGoCorr}

\subsection{Global analysis of mRNA stability changes during the
nitrogen upshift}

Previously, we found that \textit{GAP1} and \textit{DIP5} mRNAs 
are destabilized in
response to a nitrogen upshift \parencite{airoldi2016steady}. We sought to
determine if mRNA destabilization is specific to NCR transporter
mRNAs by measuring global mRNA stability across the upshift
using 4-thiouracil (4tU) labeling and RNA-seq 
\parencite{neymotin2014determination,munchel2011dynamic}.
As 4tU labeling requires nucleotide transport, which may be altered
upon a nitrogen-upshift \parencite{hein1995npi1}, we designed experiments such
that following complete 4tU labeling and metabolism to nucleotides 
the chase was initiated prior to addition of glutamine or water (mock).
We normalized data using \textit{in vitro} synthesized thiolated 
spike-ins by fitting a log-linear model to spike-in counts
across time (\nameref{subsection:bff}), which reduced noise and increased
our power to detect stability changes (\nameref{itm:dme211filterModel}).
Data and models for each transcript can be visualized in browser
using a Shiny appplication (see
\url{http://shiny.bio.nyu.edu/users/dhm267/} or \nameref{subsubsection:codeanddata} ). 

We modeled the
log-transformed normalized signal for each mRNA using linear
regression (\nameref{itm:dme211resultsModel}).
Of 4,859 mRNAs measured we identified 94 that increased in 
degradation rate and 38 that decreased (FDR < 0.01, using
\cite{storey2003statistical}). 
We generated a model of nucleotide
labeling kinetics to assess the effect of an incomplete label 
chase on our experimental design ( \nameref{subsec:4tuNormalization} ),
 and found that complete transcriptional inhibition alone could 
theoretically result in a 17\% increase in the apparent 
degradation rate. In order to eliminate the possibility that
rapid synthesis changes could affect our estimates,
we only considered destabilization of at least a
doubling (100\% increase) of apparent degradation rates between 
pre-upshift and post-upshift.
This conservative cutoff 
left 78 mRNA that are significantly destabilized 
upon a nitrogen upshift. 

The vast majority of transcripts (4,781 of 4,859) do not show
individual evidence for stability changes upon addition of glutamine
(e.g. \textit{HTA1}, \autoref{fig:figure2a}). 
The median pre-upshift half-life is 6.92 minutes and the median
half-life following the upshift is 6.32 minutes (\autoref{tab:table1})
suggesting that there is not a global change in mRNA stability.
Global stability estimates are
considerably lower than previous estimates in rich medium
\parencite{munchel2011dynamic,neymotin2014determination,miller2011dynamic}, 
which may reflect the
different nutrient conditions used in our study. 
The 78 transcripts significantly destabilized upon the 
glutamine-upshift include
mRNAs encoding NCR transporters \textit{GAP1}, \textit{DAL5}, and
\textit{MEP2} (blue label, \autoref{fig:figure2a}), the pyruvate metabolism enzymes
\textit{PYK2} and \textit{PYC1} (orange label), and trehalose synthase
subunits \textit{TPS1} and
\textit{TPS2} (yellow label).
Destabilized mRNA tend to be more stable before the upshift
(\autoref{fig:figure2bc}),
(median half-life of 9.46 minutes) and exhibit 
a median 3.06-fold increase in degradation rates (median half-life of
3.02 minutes following the upshift). 

\label{subsection:stabilityChanges}

\vspace{2em}

\begin{table}[h]
\small
\caption{\label{tab:table1} Summary of mRNA stability, median values}
\begin{tabular}{p{8em} | l | l | l | l | l | l}
%\toprule
& \multicolumn{2}{c}{Pre-shift} & \multicolumn{2}{c}{Post-shift} &
Change in & Fold-change\\
 & specific & half-life & specific  & half-life & specific & specific \\
 & rate & & rate & & rate & rate \\
 & (min$^{-1}$) & (min) & (min$^{-1}$) & (min) & (min$^{-1}$) & \\
\midrule
\raggedright All transcripts & 0.100 & 6.92 & 0.110 & 6.32 & 0.00865 & 1.08\\
\midrule
\raggedright Destabilized (n=78) & 0.0732 & 9.46 & 0.229 & 3.02 & 0.158 & 3.06\\
\midrule
\raggedright Unchanged (n=4781) & 0.101 & 6.89 & 0.108 & 6.40 & 0.00728 & 1.07\\
\bottomrule
\end{tabular}
\end{table}

\afig{
  \includegraphics[width=\textwidth]{img/Figure2a.png}
  }{
    4tU-labeled mRNA from each gene was measured over time, before and
    after the addition (vertical dotted line) of glutamine 
    (nitrogen-upshift) or water (mock). Linear regression models were 
    fit to the data with a rate before the upshift (solid line) 
    and a rate after glutamine addition (dashed line). 
    \textit{HTA1} is not significantly destabilized, 
    whereas mRNAs encoding NCR-regulated transporters 
    (\textit{GAP1}, \textit{DAL5}, \textit{MEP2}) or 
    pyruvate (\textit{PYK2}, \textit{PYC2}) and 
    trehalose (\textit{TPS1}, \textit{TPS2}) 
    metabolism enzymes are significantly destabilized. 
    \label{fig:figure2a}
  }{4tU label-chase RNA sequencing measures mRNA stability changes 
    following a nitrogen upshift.}

We tested for
functional enrichment among the set of 78 destabilized
mRNAs and found that they are strongly enriched for NCR
transcripts (16 of 78, p < $10^{-11}$). Almost half of the
destabilized transcripts are annotated as “ESR-up” genes
(\autoref{fig:comparisonESR}), on the basis of  their rapid induction
during the environmental stress response \parencite{gasch2000genomic}. These 78
destabilized mRNA are enriched (FDR < 0.05) for GO terms and KEGG 
pathways (\nameref{itm:dme211goAndKegg}) including
glycolysis/gluconeogenesis (6 genes), 
carbohydrate metabolic process (24),
trehalose-phosphatase activity (3), 
pyruvate metabolic process (6), 
and secondary active transmembrane transport
(8, a subset of the enriched 11 ion transmembrane transport genes).
%We also see destabilization of \textit{PYK2} and \textit{HXK1},
%both of which are isozymes expressed highly in poor nutrient conditions.
Thus destabilized mRNA upon a nitrogen upshift regulates, 
but is not restricted to, NCR-regulated mRNA and reflects broader
metabolic changes in the cell. 

\label{subsec:destabGoCorr}

To investigate the extent to which mRNA stability changes contribute
to transcriptome reprogramming, we compared degradation rates
to abundance changes following the upshift 
(\cite{airoldi2016steady}, \autoref{fig:figure2bc}). 
Changes in mRNA degradation rates
and mRNA abundance change rates are anti-correlated (Pearson's $r$ = -0.598,
p-value < $10^{-15}$, \autoref{fig:kkdComparison}),
consistent with stability changes impacting mRNA abundance dynamics.
However, they are not entirely co-incident, as some destabilized
transcripts do not exhibit decreases in abundance (red points in
\autoref{fig:figure2bc}, \autoref{fig:comparisonDestabilized},
and \autoref{fig:compareSix}).
This analysis shows that increases in degradation rates play a 
significant role
in the rapid reprogramming of the transcriptome upon a glutamine
upshift, but that in some cases they are counteracted by
increases in mRNA synthesis rates
\parencite{shalem2008transient,canadell2015impact}.

\afig{
  \includegraphics[width=\textwidth]{img/Figure2bc.png}
  }{
    Comparison between the pre-upshift mRNA
    degradation rate (y-axis) and the post-upshift mRNA degradation rate
    (x-axis). Negative values result from noise on the slope estimate.
    Comparison between changes in mRNA abundance following
    upshift \parencite{airoldi2016steady} (y-axis) and the post-upshift
    degradation rate (x-axis). Both plots share the same x-axis.
    Transcripts that are significantly destabilized are colored red, and
    shown with red rug-marks in the marginal histograms.
    \label{fig:figure2bc}
  }{Global mRNA stability changes following a nitrogen upshift.}

\afig{
  \includegraphics[width=\textwidth]{img/Figure2_S_globalComparisons.png}
  }{
  Pre-upshift decay rates \textbf{(top)} don't explain the
  abundance change. 
  The degradation rate changes (\textbf{middle}, difference between pre
  and post upshift) and the post-upshift rates \textbf{(bottom)} 
  are anti-correlated with the abundance changes.
  \label{fig:kkdComparison}
  }{Comparison between rates of mRNA degradation and abundance
    changes.}

\afig{
  \includegraphics[width=\textwidth]{img/Figure2_S_comparisonToESR.png}
  }{
  Comparisons of degradation rates from this study with mRNA abundance change rates
  from \cite{airoldi2016steady}. Destabilized transcripts are colored based on
  their membership in the ESR gene set \parencite{gasch2000genomic}, 
  as described in the supplement 
  of \cite{brauer2008coordination}. 
  Many of the destabilized set are ESR "up" genes, as they
  are increased in expression in response to stresses.
  \label{fig:comparisonESR}
  }{Many of the destabilized mRNA are members of the ESR-up regulon.}

\afig{
  \includegraphics[width=\textwidth]{img/Figure2_S_justDestabilizedDecayvsDynamics.png}
  }{
    For each, the x-axis is
    the fit rate of degradation rate post-upshift. On the y-axis is the mRNA abundance
    change rate \parencite{airoldi2016steady} after the upshift.
    These values were modeled to normalized
    sequencing signal (x-axis) and normalized microarray ratio (y-axis). The dashed
    line is a 1:1 line of equality.
    \label{fig:comparisonDestabilized}
  }{Significantly destabilized transcripts are not always strongly
    repressed.}

\afig{
  \includegraphics[width=\textwidth]
    {img/Figure2_S_sixExamplesOfDestabilizationWithoutRepression.png} 
  }{
    For several examples of the slowest decreasing (in the microarray fits)
    transcripts, we plot the microarray (abundance) and sequencing (decaying labeled
    abundance) data normalized to be on the same relative y-axis scale (subtracted
    $t_0$ y-intercepts of fits).
    Destabilization does not necessarily result in a rapid clearance
    of the mRNA.
    \label{fig:compareSix}
  }{Six examples of individual mRNA whose regulation is more
    complex than a homo-directional destabilization and synthesis
    repression.}

Functional coordination of mRNA stability changes suggests  a possible
role for \textit{cis}-element regulation. We analyzed UTRs and coding
sequence for enrichment of new motifs or known RNA binding protein
(RBP) motifs.
3' UTRs of destabilized transcripts are
depleted of Puf3p binding sites, and we found no enriched sequence
motif in the 3' UTRs.
5' UTRs are enriched for a GGGG motif, which
may be explained by convergence between mRNA stability changes and
transcriptional control by Msn2/4 on the ESR “up” genes
(\autoref{fig:comparisonESR},
\cite{gasch2000genomic,canadell2015impact}). 
5' UTRs are also enriched for binding motifs reported for Hrp1p 
(\autoref{fig:hrp1}),
a canonical member of the nuclear cleavage factor I complex
\parencite{chen1998specific}.
However, this protein has been shown to shuttle to the cytoplasm
where it may play a regulatory role
\parencite{kessler1997hrp1,kebaara2003upf,guisbert2005functional}.
On average,
destabilized mRNAs are longer and contain more optimal codons
(\autoref{fig:lengthAndCodons}, \cite{khong2017stress}). 
Together, this analysis suggests that the
mechanism of destabilization may act through cis elements in the 5'
UTR or biased codon usage.
Both of these may affect the density or elongation kinetics of the
process of translation, which may directly interact with components
involved in mRNA degradation or indirectly affect the accessibility
of the mRNA to these components in the cytoplasm \cite{huch2017mrna}.

\afig{
  \includegraphics[width=.8\textwidth]{img/Figure2_S_averageMotifsPerSection.png}
  }{
    Sequences of destabilized and unaltered mRNAs were analyzed for
    RBP binding motif enrichment
    using the AME program in MEME, then significant hits were confirmed by using a
    logistic model predicting destabilization based on motif content per sequence
    length. Hrp1p is significantly ( p<0.0001 ) enriched in
    the 5' UTRs of destabilized transcripts. For this plot, motif matches were
    counted using the GRanges package \parencite{lawrence2013software}
     for the 5' UTRs, 3' UTRs,
    and coding sequence of transcripts using the largest isoforms detected in
    \cite{pelechano2014genome}.
    \label{fig:hrp1}
  }{Enrichment of Hrp1p motif in 5' UTRs of destabilized transcripts.}

\afig{
  \includegraphics[width=.8\textwidth]{img/Figure2_S_lengthAndCodons.png}
  }{
    Comparisons of destabilized mRNAs with the rest of the transcriptome.
    \textbf{(Left)} 
    Destabilized transcripts tend to have longer CDS lengths ( p-value < 2e-5
    by Wilcoxon rank sum test ). 
    \textbf{(Right)} 
    The destabilized transcripts tend to have a greater fraction of
    codons that are considered optimal codons
    than the non-destabilized transcripts 
    ( p-value < 2e-8 Wilcoxon rank sum test).
    The fraction of optimal codons per feature
    was obtained from the supplement of \cite{khong2017stress} using definitions
    from \cite{presnyak2015codon}. 
    \label{fig:lengthAndCodons}
  }{The destabilized set is longer and has a higher frequency
    of optimal codons than the rest of the transcriptome.}

\subsection{Methods and materials}

\subsubsection{Measurement of growth during upshift}

A single colony of FY4 was inoculated in 5mL proline-limited
media and grown to exponential phase, then back diluted in
proline-limited media
in a baffled flask. 
Samples were collected into an eppendorf, sonicated,
diluted in isoton solution, and analyzed with a Coulter Counter Z2
(Beckman Coulter).

\subsubsection{Re-analysis of microarray data} 

Supplemental files from \cite{airoldi2016steady} were 
downloaded, read
into \texttt{localc} (an open-source spreadsheet software), 
a small Excel-generated auto-correction error was 
fixed ("Oct-1" -> "OCT1"), and the file saved as a CSV. Microarray 
intensity ratios were processed with 
\texttt{pcaMethods} to perform a SVD PCA on scaled data. 

\subsubsection{Synthetic RNA spike-in generation}

\label{subsubsection:spikeins}

Poly-adenylated RNA molecules were synthesized \emph{in vitro} using a
Promega Riboprobe SP6 kit (P1420) with 4-thiouridine, as previously
described \parencite{neymotin2014determination}, to serve as
spike-in calibrators for RNAseq normalization across samples.
Ampure XP beads were used to clean up the reaction.
Products were quantified using the Qubit HS RNA assay, and 
equivalent mass amounts of spike-ins were pooled to create a 
8ng/\(\mu\)L stock containing all four 4-thiouridine-labeled 
spike-ins.

We also prepared total 4-thiouracil labeled \emph{E. coli} RNA to 
use as another spike-in. We grew strain MG1655 (a gift of Edo Kussell)
overnight in 5mL of LB with 20\(\mu\)M of 4-thiouracil, then outgrew
this for 2.5 hours in the presence of 4-thiouracil, then extracted
by boiling the pellet with 1\% SDS, 100mM NaCl, and 8mM EDTA for 5
minutes with intermittent vortexing, before cleaning up with a phenol
chloroform extraction. The product was quantified using qubit and 
diluted to a 5ng/\(\mu\)L solution of thiolated total \emph{E. coli} 
RNA.

\subsubsection{Culturing and sampling}

FY4 was grown in nitrogen-limitation conditions overnight (26 hours) 
with a mixture of 50\(\mu\)M:50\(\mu\)M of 4-thiouracil:uracil. 
The culture was split into two 450ml cultures 5 hours before
the label chase began. During exponential phase growth ( \(\sim\) 5
\(\times 10^6\) cells per mL), uracil from a 400mM DMSO stock was added
to a final concentration of 4mM (41-fold excess) to chase the label.
30mL samples from the culture were collected by filtration and
flash-freezing within a minute of removal from the flask, and 
sampling time is recorded as the time of flash-freezing. 
After letting the chase proceed for 12.5 minutes, I added glutamine
from 100mM stock (dissolved in water) to a final concentration of
400\(\mu\)M to one flask, or an equal volume of water to the control
flask.

Timepoints were chosen to sample five times before the intervention,
but timepoints actually used are the times that the sample was dropped
into liquid nitrogen for fixation.
For mock treatment (water at 13 minutes), this was 
3.85, 6.02, 7.92, 9.90, 11.8, 15.1, 17.0, 18.8, 20.8, 22.9, 26.1, 
and 50.5 minutes. 
For nitrogen upshift (glutamine at 12.5 minutes), this was
3.30, 5.32, 7.65, 9.47, 11.3, 14.4, 16.4, 18.2, 20.0, 23.8, 28.8, 
and 49.1 minutes.

\subsubsection{RNA Extraction}

Since equal volume (30mL) of culture was taken for each sample, an equal
volume of synthetic spike-ins was added to each RNA extraction reaction
(hot acid-phenol method).
The extraction yielded at least 3.3 \(\mu\)g of RNA per \(10^7\) cells.

\subsubsection{Biotinlyation and fractionation}

The total RNA (yeast and spike-ins, mixed) was reacted by adding
a total of 20$\mu$g MTSEA-biotin diluted in DMF to
200$\mu$L of HEPES buffered total RNA solution, 
and incubating for 2 hours.
Biotinylated total RNA was fractionated with streptavidin bead
pulldown, using 200\(\mu\)L of NEB (S1420S) streptavidin beads. 
These were bound for 20 minute incubation, then pulled down and
solution discarded, then washed four times with bead buffer, three
times at room-temperature and once at 65C. Labeled mRNA was eluted
using beta-mercaptoethanol, twice, then precipitated.

\subsubsection{rRNA depletion}

Fractionated RNA was depleted of rRNA using the RiboZero kit (Illumina
RZY1324) according to manufacturer instructions, except that for
every reaction of 2\(\mu\)g input RNA, we used half-reactions 
(half of every supplied reagent). 
Final RNA was ethanol precipitated, as above. Agilent
Tapestation measurements of the RNA size histograms confirmed that
virtually all of the rRNA was removed, and later computation analysis
found very few rRNA reads in the sequencing results.

\subsubsection{Preparing sequencing libraries}

RNA samples were converted into Illumina sequencing libraries using a
strand-specific (UNG) protocol.  Briefly, 1st strand cDNA was 
synthesized using a SuperScript III kit
(Invitrogen), primed with random hexamers. RNA was fragmented by
98C hybridization for 1 minute in 4.17mM MgCl$_2$, then actinomycin
and the SuperScriptIII enzyme were added after annealing.  
This was incubated to extend first strand in a series of increasing
temperature steps over the course of an hour.
This reaction was ethanol precipitated
and resuspended for a second-strand reaction that incorporated dUTP
in the place of dTTP, using a cocktail of DNA PolI, Ecoli ligase, and
RNAseH. This was reacted at 16C for 2 hours, then cleaned up on
MinElute nucleic acid purification columns (Qiagen 28004).
This product was eluted, end-repaired using T4 DNA polymerase and PNK,
then cleaned up on MinElute columns.
This product was eluted, A-tailed with Klenow (exo-), 
then cleaned up on MinElute columns.

This product, fresh from A-tailing, was ligated with "TrUMISeq"
adapters made by a former graduate student in the lab
\parencite{hong2017method}. These are TruSeq adapters, but the
sample index has been incorporated as the first six bases sequenced
from the sequencing priming site and a final T exists to help with
ligation. In place of the sample index (interior to the adapter),
a degenerate sequence is incorporated during synthesis.
These adapters, theoretically, mark each unique ligation event
with a DNA barcode sampled from a degenerate pool of $4^6 = 4096$
different barcodes. This greatly reduces the chance that two
molecules that appear to be PCR duplicates (false double-counting
by virtue of the amplification scheme) are actually considered
to be duplicates in the analysis. For a more in-depth discussion of
UMIs, please see
\nameref{subsubsection:rarefractionDiscussion}.

These ligations were all of the A-tailed product with 20nM annealed
adapter, each reaction with a different sample index. These were
reacted using Quick Ligase (NEB), reacted at 22C for 15 minutes
before immediate clean-up with Ampure XP beads (BeckmanCoulter).
These were selected twice on beads to remove small adapters.
To amplify libraries and select the strand-specificity, I prepared a
master-mix of NEB Phusion buffer with primers DGO366 and DGO367,
reacted this with UNG (Thermo EN0361) to digest the dUTP containing
strand, then added NEB Phusion polymerase and PCR amplified for
18 cycles.
These reactions were cleaned up using a MinElute column, then diluted
and concentration estimated using qPCR on a Roche 480 (using KAPA
Library Quant Kit Illumina REF 07960281001), and submitted as a 1nM pool
to the NYU GenCore system for sequencing on a NextSeq using the 75bp
format in High-Output mode.

\subsection{4tU label-chase sequencing analysis and modeling}

\subsubsection{Quantifying sequencing reads}

Following base-calling and demultiplexing by NYU GenCore, the sequencing
reads were quantified using the following pipeline:

\begin{enumerate}
  \setlength\itemsep{0em}
  \item Raw reads were trimmed using \texttt{cutadapt} \parencite{martin2011cutadapt}
  \item Trimmed reads were aligned using \texttt{tophat2} \parencite{kim2013tophat2}
    to a reference genome that included the yeast reference 
     genome (assembly R64), the Ecoli genome (assembly 
     GCF\_000005845.2), and the four synthetic in-vitro transcribed 
     spike-ins (termed BES and available in the \texttt{data.zip} 
     archive of the OSF archive associated with this paper). 
     This was done with parameters optimized 
     against \emph{in silico} data generated by Flux Simulator
     \url{http://sammeth.net/confluence/display/SIM/Home} from this 
     reference genome, in replicates.
  \item Reads with mapping quality above 20 and with at least 50 
    matched bases with 
    \texttt{samtools} \parencite{li2009sequence}
    were processed with \texttt{umi\_tools} \parencite{smith2017umi}
    in ``dir'' mode to
    de-duplicate possible PCR duplicates.
  \item The demultiplexed \texttt{.bam} file was processed with the
    \texttt{htseq-count} \parencite{anders2015htseq}
    script to generate counts files per gene feature (according to 
    the GFF file in the \texttt{data/BES} directory).
\end{enumerate}

\subsubsection{Normalization of counts into signal for modeling}

\label{subsec:4tuNormalization}

In order to accurately estimate degradation rates, I must accurately
estimate labeled mRNA abundance within each timepoint. This is
achieved by normalizing the signal of the mRNA (counts) to the signal
of a spiked-in reference transcript pool (the four spike-ins added
during extraction).

The simplest normalization is to divide each feature counts
by the sum of the counts of all the spike-ins (personal communication,
Daniel Tranchina). However, the low RNA abundances of several 
samples in this experiment had poor quantification of the spike-in 
which required me to remove outlier measurements to prevent 
systematically noisy data from
disrupting the quantification. 

I decided to smooth the normalization across the samples, as we expect
the proportion of counts that are the spike-ins to increase over time.
This assumes that the whole transcriptome decays with exponential
kinetics \parencite{petersen1976half}.

In the sequencing data, we clearly see that the proportion of counts
that are the spike-ins increases with time.
\autoref{fig:propIncrease}.
We modeled this increase using the \texttt{lm} function to linearly
regress this in natural-log-space. We see that
the residuals are randomly distributed around the fit across time 
for both treatments \autoref{fig:propIncrease}.
Using an ANCOVA (\texttt{aov}/\texttt{lm}), I found the effect of 
treatment was associated with a p-value < 0.301 and the p-value
associated with time estimated as ``1'', so it does not appear that the
residuals particularly depend on time or treatment.

\afig{
  \includegraphics[width=.49\textwidth]{img/fig2s_increase_props.png}
%  \includegraphics[width=.49\textwidth]{img/fig2s_increase_residuals.png}
  }{
  The observed proportion of counts that are synthetic spike-ins
  is each point. The line is a regression fit to both treatment 
  datasets.
  \label{fig:propIncrease}
  }{The proportion of spike-in counts increase over the course of the
    experiment.}

How do the normalizations compare on a per-gene basis? 
\ref{fig:normgenes} compares this.


\afig{
  \includegraphics[height=6in]{img/fig2s_norm_example.png}
  }{
  The results of the two strategies are shown on the left (direct
  normalization) and on the right (model-based). For several example
  genes, we see that the model-based smoothing reduces the noise.
  \label{fig:normgenes}
  }{Several examples of alternative normalization strategies.}

The direct normalization of each gene in each timepoint combines the 
noise of both the measured gene and the measured spike-ins. 
By smoothing across timepoints, 
given our expectation of whole-transcriptome exponential
decay dynamics, we can deliver a more reliable estimate for each gene
feature. Thus more observations can be reliably analyzed
without single sample spike-in errors systematically skewing the fits
across all features.

We also tried to spike-in labeled \textit{E. coli} total RNA; 
however, we found
that those counts were low, noisy, and did not behave as expected
from previous work \parencite{neymotin2014determination}. We
hypothesize that this was due to lower addition of \textit{E. coli} 
total RNA than synthetic spike-ins, 
combined with noise associated with amplifying a
random sub-sample of a more complex spike-in pool of total \textit{E.
coli} RNA. Each sub-sample may have very different GC-content, and
thus may be amplified to a different degree.
Thus, we normalized all yeast mRNA to the synthetic spike-ins as
previously demonstrated.
Sampling from a complex spike-in library may be an important
consideration for their use in normalization, and may warrant further
technical experimentation.

\subsubsection{Model of transcript dynamics as a function of 
dynamics and labeling parameters}

To analyze this data, I fit a model of 
labeled transcript dynamics.
I used this model to analyze the dataset for expected label-chase
dynamics, and also to exclude effects that may result from a
confounding of new synthesis with changes in degradation rates.

\(m_t\) is the labeled mRNA at time \(t\). It changes according to the
equation: \[ \frac{d m_t}{dt} = L k_s - k_d m_t\] where \(L\) is the
fraction of new mRNA that is labeled and pulled down, \(k_s\) is the
rate of synthesis, and \(k_d\) is the rate of degradation. Our
experimental design is to change \(L\) from an initial fraction of
transcripts that are pulled down by a 4tU-incorporation-dependent
mechanism of \(L^o\) (old) to a new fraction \(L^n\) (new). Note that
I use the notation as a superscript, so that I can also share that
notation with the synthesis rates as \(k_s^o\) and degradation rates as
\(k_d^o\).

I assume that the culture begins at a steady state of
\(L^{o}\frac{k_s^{o}}{k_d^{o}}\), from solving the above equation.
I assume this because I grow the cells for 24 hours in conditions
of labeling, and they are well below saturating conditions.
Solving the above differential equations with the assumption that
everything changes once, which is a simplifying assumption but supported
by previous studies of transcript stability changes during shifts
\parencite{perez2013eukaryotic}, I expect that \(m_t\) should behave
as, \[ m_t = L^o \frac{k_s^o}{k_d^o} e^{-k_d^n t} + 
  L^n\frac{k_s^n}{k_d^n}(1-e^{-k_d^n t}) \] Nicely, the solution is
similar to what I would expect intuitively - extant transcripts decay
(left), and nascent transcripts approach the new equilibrium (right).
The equilibriums are set by all parameters, but the change between them
is dictated by the new degradation rate operative during the transition.

In the case were either \(L^o\) or \(L^n\) is 0, then the transcript
behaves just as one side of the equation. With the label-chase, I are
trying to get \(L^n\) as low as is possible without perturbing the
system being measured by killing the cell.
To analyze this dataset for potential changes in transcript stability,
I approximated this by fitting a linear regression model to the
normalized signal. I explore the sufficiency of this model later in
this document using simulations. This model was fit using the
\texttt{lm} function in R, with the formula

\begin{verbatim}
log( NormalizedSignal ) ~ Minutes + Minutes:Treated + 1
\end{verbatim}

where ``\texttt{NormalizedSignal}'' is the signal of the gene feature
normalized as described in the previous section, ``\texttt{Minutes}'' is
minutes relative to the glutamine (or water) addition,
\texttt{Minutes:Treated}'' is an additional slope of the observations
after glutamine addition, and ``\texttt{+\ 1}'' denotes to fit a single
intercept for the model (data are centered around the moment of 
treatment, $t_0$ is the addition of glutamine or water). 
From this fit, I took the p-values associated
with the t-statistic of the additional slope fit to the glutamine
treated samples, then adjusted the p-values using the \texttt{qvalue}
package from BioConductor using default settings. I chose to use a FDR
cut-off of less than 0.01 for this analysis.

\subsubsection{Estimating possible effects of synthesis changes on 
labeled abundance}

In our experimental design I initially grow the cells in a
50\(\mu M\):50\(\mu M\) mix of uracil and 4-thiouracil, so I will set
as a labeling ratio \(L^o\) of 1 for simplicity. I add 4,000 \(\mu M\)
uracil to begin the chase, so this is a shift to a \(L^n\) of
\(\frac{50 \mu M}{4100 \mu M} / \frac{50 \mu M}{100 \mu M}\) , or
\(\frac{1}{41}\). Since I am not reducing this number to zero, there
is still residual labeling incorporated into nascent transcription.
\(\frac{1}{41}\) is a small number, but is still not zero and should
not be neglected.
Thus, there is a potential that residual label could confound our
estimate of degradation rates. This is an inherent tradeoff in a
label-chase design \parencite{perez2013eukaryotic}, 
especially since the low RNA content of the cells
and low cell density in these nitrogen limited conditions make necessary
the use of a more efficient pull-down reagent (MTSEA-biotin). This could
be circumvented by comparing abundance and synthesis measurements, but
the uracil transporter responding to glutamine in the media makes this
technically difficult with 4tU incorporation. Comparing abundance and
mRNA synthesis by other means is feasible, but introduces a compounding
of errors from both methods. Thus performing one direct assay is
preferable for precision, and the drawbacks described here are
unavoidable with current technology (although progress is being made
\parencite{chan2017non}).

Therefore, I used simulations to investigate how varying the labelling
parameter changes the expected dynamics if we also vary the synthesis
parameter. \autoref{fig:modelingNoChange} (left) shows a plot of the modeled
labeled transcript abundance, with no change in synthesis parameter.
How does this estimate of change in degradation look if we decrease the
\(k_s\)? For example, the NCR regulon is expected to be shut-off at the
synthesis level quickly upon glutamine addition, so how would that swift
repression affect the apparent change in labeled mRNA dynamics?
\autoref{fig:modelingChange} (right) shows a plot of the modeled
labeled transcript abundance, with a $k_d$ of 0.1 (similar to the
median rate of degradation \autoref{fig:figure2bc}) 
and an instantaneous shutoff of transcript synthesis.

\afig{
  \includegraphics[width=.49\textwidth]{img/fig2s_no_change_synth.png}
  \includegraphics[width=.49\textwidth]{img/fig2s_shutoff_synth.png}
  }{
  The timepoints along the bottom are the exact timepoints used in
  the actual label-chase RNA sequencing experiment.
  \textbf{(Left)} is with no change in synthesis.
  \textbf{(Right)} is with a complete halt in synthesis. 
  We see a 13.3\% decrease in the slope of change in labeled
  abundance over time.
  \label{fig:modelingNoChange}
  \label{fig:modelingChange}
  }{
    Simulation of labeled transcript dynamics shows a slight effect
    from synthesis changes on a label-chase with recycling.
  }

The effect of reduced synthesis on apparent slope change of the 
labeled RNA here is a 13.3\% increase in the rate.
I conclude that there's a slight effect from synthesis rate changes.
For transcripts that are putatively stabilized, the effect is 
much larger ($\sim$60\%), likely because of the delay in pertubation
until 13 minutes after the label chase begins.

What does this mean for our estimates of destabilization? What effect
sizes are estimated, and how do they compare to this effect size?
\autoref{fig:changeRateDist} shows the distribution of the fold changes in
stability:

\afig{
  \includegraphics[width=.7\textwidth]{img/fig2s_distribution_of_rates.png}
  }{
  Histogram of the fold-changes in rate of labeled transcript
  clearance upon glutamine treatment, determined as significant by 
  adjusting the parametric (t-statistic of slope) with a
  multiple-hypothesis correction (\texttt{qvalue} package
  \cite{storey2015qvalue}). The blue line denotes a 13.3\%
  increase in the rate of clearance, the red line denotes a doubling
  of the rate. Some transcripts appear stabilized (left of 0), most
  are destabilized, many are destabilized more than the red cutoff.
  \label{fig:changeRateDist}
  }{Distribution of all fit changed rates versus effect size 
    thresholds from modeling.}

I see that all of the significant changes are in great excess to that
blue line. I choose a strict cut-off of a 100\%
increase, a doubling, of apparent degradation rate to call a feature
destabilized (right of the red line). Since I cannot place an upper
bound on the synthesis rates after a glutamine upshift, I cannot
definitively say that the potentially stabilized transcripts (left of 0)
are stabilized without additional experiments.
Could these fits just be on the right side of the blue line by chance?
Given that the t-statistics (ratio of effect size over standard error)
for the fits of ones over this line are a median of -5.66,
I'm not going to have fits within several standard errors of
crossing that threshold by a reasonably expected error.

I conclude that the RNA from 78 gene features are degraded 
more quickly than can be reasonably
explained by labelling carry-over, and are thus accelerated in
degradation upon the nitrogen upshift.
Stability changes upon the nitrogen upshift generally exhibited the  
expected relationship with rates of abundance change (               
anti-correlation, $R^2=$-0.376 ).
However, I caution that this data is not suitable for direct
interpretation with regards to the fraction of transcriptome
reprogramming attributable to degradation rate changes, 
as an unknown fraction of the correlation could be explained
by large changes in synthesis rate systematically confounded into
the degradation rate estimate.
I encourage the interested reader to go to
\url{http://shiny.biology.nyu.edu/users/dhm267} to explore examples of
the data under both normalizations and for a range of features.

\subsubsection{\textit{Cis} element analysis}

To detect if \textit{de novo} or known \textit{cis} elements were 
associated with destabilization upon a nitrogen upshift,
I used a variety of bioinformatic methods.  
For each transcript, I used a GFF file to extract the coding sequence
of each annotated mRNA and four different definitions of its
untranslated regions --- 
200bp upstream of the start codon or downstream
of the stop codon, the largest detected isoform in TIF-seq from
\parencite{pelechano2014genome}, 
or the most distal detected gPAR-CliP sites in
exponential-phase or nitrogen-limited growth in 
\parencite{freeberg2013pervasive}.
A more rigorous definition using 5' and 3' end sequencing methods 
in this particular condition would be necessary for best exploring 
\textit{cis}-element enrichment with certainty. 

To find putative cis-elements, 
I used DECOD \parencite{huggins2011decod}, 
FIRE \parencite{elemento2007universal},
TEISER \parencite{goodarzi2012systematic},
and the \#ATS pipeline \parencite{li2010predicting},
I also scanned for RBP binding sites from
CISBP-RNA \parencite{ray2013compendium}
using AME from the MEME suite \parencite{mcleay2010motif}.

The enrichment for the Hrp1p motif is described in \autoref{fig:hrp1}.
It is worth noting that this is the core-element of Hrp1p (also known
as Nab4p) described in \cite{guisbert2005functional,chen1998specific}, 
but is mistakenly recorded in the CISBP-RNA database as being the 
motif for Npl3p. 
\cite{guisbert2005functional} did not find a motif for Npl3p.

%%%%%
%%%%%
%%%%%
%%%%%
%%%%%

\section{Estimating \textit{GAP1} mRNA for every mutant in a pool}

\subsection{A genome-wide screen for \textit{trans}-factors 
  regulating \textit{GAP1} mRNA repression}

We sought to identify \textit{trans}-factors mediating accelerated mRNA
degradation in response to a nitrogen upshift. We selected \textit{GAP1} 
as representative of transcript destabilization, as it is abundant in
nitrogen-limiting conditions and is rapidly cleared upon addition of
glutamine  (3.24-fold increase in degradation rate, \autoref{fig:figure3a},
\nameref{itm:dme211resultsModel}). Previous approaches to high-throughput
genetics of transcriptional activity have used protein expression
reporters \parencite{neklesa2009genome,sliva2016barcode} or automation of qPCR 
\parencite{worley2016genome}. However, for our
purposes, we required direct measurement of \textit{GAP1} mRNA 
changes on a rapid timescale.
We decided to focus on a single mRNA readout to explore the efficacy
of this approach. 
While multiple readouts is theoretically more informative, the
practicality of expanding the SortSeq paradigm to another marker
poses technical challenges beyond our current capabilities
(moving from 4 bins to 16 bins dilutes the library greatly,
available sorter does 4 bins maximum).
Therefore, we applied single molecule fluorescent \textit{in situ}
hybridization (smFISH) to quantify 
native \textit{GAP1} transcripts in yeast cells in the pooled
prototrophic yeast deletion collection \parencite{vandersluis2014broad}.
Using fluorescence activated cell sorting (FACS) and Barseq
\parencite{smith2009quantitative,robinson2014design,giaever2014yeast},
we aimed to quantify and model the distribution of \textit{GAP1} mRNA
in each mutant \parencite{kinney2010using,peterman2016sort}.

\afig{
  \includegraphics[width=.6\textwidth]{img/Figure3a.png}
  }{
  \textit{GAP1} mRNA following upshift measured using RT-qPCR, 
  relative to an external spike-in mRNA standard. The dashed line 
  is fit to points after 2 minutes. 
  \label{fig:figure3a}
  }{\textit{GAP1} mRNA dynamics by qPCR normalized to a spike-in shows
    the rapid clearance of \textit{GAP1} mRNA.}


%Development of our screen required that we could detect and
%sort cells using \textit{GAP1} mRNA signal. 
We found that
individually labeled probes tiled across \textit{GAP1} mRNA
\parencite{raj2008imaging} were insufficiently bright for
\textit{GAP1} mRNA quantification using flow cytometry (data not shown),
likely due to the small cell size of nitrogen-limited cells and the
low transcript numbers in yeast cells compared to mammalian cells
\parencite{klemm2014transcriptional}. Therefore, we used branched DNA probes
(Quantigene), which serve to amplify the FISH signal
\parencite{hanley2013detection}. We developed a fixation and permeabilization
protocol (\nameref{subsection:bff}) that enabled detection of the
distribution of  \textit{GAP1} mRNA in steady-state nitrogen-limited conditions
and its repression following the  upshift (\autoref{fig:figure3b}). In control
experiments, we found that the signal is eliminated in a \textit{GAP1} deletion
or by omitting the targeting probe % that confers specificity
(\autoref{fig:figure3b} and \autoref{fig:gap1Delete}). To validate
sorting, we sorted a sample of cells into quartiles and used
microscopy to count fluorescent foci per cell
(\autoref{fig:figure3c}) .
We found that increased flow cytometry signal is associated with an
increase in the number of foci in the cells (\autoref{fig:figure3d}, $R^2$ = 0.607,
p < $10^{-11}$ ). 

\afig{
  \includegraphics[width=.7\textwidth]{img/Figure3_S_gap1deleteControl.png}
  }{
  Wild-type or \textit{gap1}$\Delta$ cells were grown in 
  proline-media, which induces expression of \textit{GAP1}. 
  As seen in the positive control, there is heterogeneity in the
  induction. This is likely due to technical issues, namely over
  fixation. Importantly, \textit{gap1}$\Delta$ does not have any
  positive signal.
  \label{fig:gap1Delete}
  }{Both the \textit{GAP1} delete and omission of the targeting 
    probe removes signal of GAP1 FISH.}

\afig{
  \includegraphics[width=.7\textwidth]{img/Figure3b.png}
  }{
  Flow cytometry of wild-type yeast in nitrogen-limited conditions 
  and following an upshift. The vertical grey lines indicate FACS 
  gate boundaries used for cell sorting, for \autoref{fig:figure3c}
  microscopy.
  \label{fig:figure3b}
  }{\textit{GAP1} mRNA dynamics measured by flow cytometry.}

\afig{
  \includegraphics[width=.7\textwidth]{img/Figure3c.png}
  }{
  Representative cells from each bin sorted from the experiment in 
  \autoref{fig:figure3b}. Alexa647 signal is imaged with a Cy5
  filter set.
  \label{fig:figure3c}
  }{Example microscopy of cells sorted by \textit{GAP1} mRNA.}

\afig{
  \includegraphics[width=.7\textwidth]{img/Figure3d.png}
  }{
  Cells sorted from \autoref{fig:figure3b} were manually scored in
  z-stacks for Alexa647 foci.
  Each black dot represents the total for a single cell. 
  The mean number of foci per cell in each bin is displayed as a red point.
  \label{fig:figure3d}
  }{Quantification of microscopy of cells sorted by \textit{GAP1} mRNA.}

Previous SortSeq studies of
the yeast deletion collection have used outgrowth 
to generate sufficient material for 
Barseq \parencite{sliva2016barcode}. However, formaldehyde fixation precludes
outgrowth. We found that below approximately $10^6$ templates, the
Barseq reaction produces primer dimers
that outcompete the intended PCR product (\nameref{subsection:bff}). 
Therefore, we re-designed the
PCR reaction \parencite{robinson2014design,smith2009quantitative} to be robust for
very low sample inputs (\nameref{subsection:bff}). Our protocol
incorporates a 6-bp unique molecular identifier (UMI) into the first
round of extension to identify PCR duplicates, 
and uses 3'-phosphorylated oligonucleotides and a
strand-displacing polymerase (Vent exo-) to block primer dimer formation and 
off-target amplification. 
%We developed a bioinformatics pipeline 
%using pairwise alignment
%for per-read quality-filtering and compatibility with variable barcode
%length, and using the degenerate UMI barcodes to help account for PCR
%duplicates. 
%UMIs to identify duplicates.
Because strain barcodes are of variable lengths, 
we developed a bioinformatic pipeline to extract barcodes and UMIs 
using pairwise alignment to invariant flanking sequences.
Based on \textit{in silico} benchmarks, this
approach was robust to systematic and simulated random errors 
that can confound analysis of the yeast deletion barcodes 
(\nameref{subsubsection:codeanddata}, \nameref{subsection:bff}). 

We refer to this experimental approach as BFF (Barseq after FACS after FISH). 
We used BFF to estimate \textit{GAP1} mRNA abundance for every mutant in the
haploid prototrophic deletion collection
\parencite{vandersluis2014broad} in
nitrogen-limiting conditions and 10 minutes following the upshift. 
This approach facilitates identification of mutants with
defects in mRNA regulation at both the transcriptional and
post-transcriptional level without altering \textit{GAP1} mRNA 
\textit{cis}-elements that may affect its regulation. 
Moreover, this design enables identification of factors that 
regulate both the steady-state abundance of \textit{GAP1} mRNA and 
its transcriptional repression following an upshift.
We analyzed the deletion pool in biological triplicate
(\autoref{fig:figure4a}). We found that UMIs 
approached saturation at a slower rate than expected for random sampling,
consistent with PCR amplification bias 
(\autoref{fig:umi}), and therefore we adopted the 
correction of \cite{fu2011counting} (dicussed in depth later). 
After filtering, we calculated a
pseudo-events metric that approximates the number of each mutant sorted
into each bin. 
Principal components analysis shows that the samples are 
separated primarily by FACS bin within each
condition and biological replicates are clustered indicating that our
approach reproducibly captures the variation of  \textit{GAP1} mRNA flow
cytometry signal across the library (\autoref{fig:pca}). 

\afig{
  \includegraphics[width=.9\textwidth]{img/Figure4_S_PCAonFilteredQCdData.png}
  }{
    Each letter is the point, with the letter denoting the biological
    replicate and dashed lines connecting the technical replicates
    of each sample.
    Each color is a type of sample, from low to high gates 
    (with black denoting the input samples before sort). 
    \textbf{(Top)} The first two prinicpal components
    show the separation of gates by signal intensity, and reflects that the lower
    gates on the upshifted samples were both sampling from the same
    negative population
    (blue and red samples on far right panel, 
    lowest two bins post-upshift in \autoref{fig:figure4a}).
    \textbf{(Bottom)} A similar pattern in principal components 3
    and 4, although this highlights that bin 1 is the most distinct
    before the shift and bin 4 is most distinct after the shift.
    \label{fig:pca}
  }{Principal components analysis of the abundance estimates for
    samples captures variation along \textit{GAP1} abundance.}

\subsection{Estimating \textit{GAP1} mRNA abundance for individual mutants}

We estimated the distribution of \textit{GAP1} mRNA for each mutant by
modeling pseudo-events in each quartile as a
log-normal distribution using likelihood maximization  
(\autoref{fig:figure4b}). 
%Specifically, replicate A had a consistently lower estimate of
%\textit{GAP1} FISH fluoresence in both flow cytometry and modeling.
%Replicate C had fewer mutants sorted (\autoref{subsection:bff}), 
%reflected in the wider distribution of estimated means.
From model fits we estimated the mean \textit{GAP1} abundance for each
mutant and found that the distribution of means estimated for
3,230 strains (\nameref{itm:dme209pooledFits}, \autoref{fig:figure4c}) 
recapitulates the overall
distribution of flow cytometry signal (\autoref{fig:figure4a}). 
%To estimate \textit{GAP1}
%mRNA per strain, we used all replicate measurement to perform model
%fitting and filtered models for sufficient measurements  (at least two
%of three replicates in at least three of the four bins). We generated
% distribution estimates for 3,230 strains, and used the mean
%of each distribution as the estimate of \textit{GAP1} mRNA abundance for each
%strain (\nameref{itm:dme209pooledFits}). %added after commented out
To validate our approach we first examined
strains for which we expected to have a specific phenotype and
compared their mean \textit{GAP1} abundance to the distribution of 
\textit{GAP1} abundances for the entire population (\autoref{fig:figure4d}). 

\afig{
  \includegraphics[width=.9\textwidth]{img/Figure4b.png}
  }{
    Measurements for individual genes before and
    after the upshift. Black dashed lines indicate maximum-likelihood 
    fits of a log-normal to pseudo-events for each mutant. 
    Each color is a biological replicate.
    \label{fig:figure4b}
  }{Data and models for several individual sample genes from the BFF
    modeling.}

\afig{
  \includegraphics[width=.9\textwidth]{img/Figure4a.png}
  \includegraphics[width=.9\textwidth]{img/Figure4c.png}
  }{
    \textbf{(Top)} 
    Flow cytometry analysis of \textit{GAP1} mRNA 
    abundance in the prototrophic
    deletion collection before and after the upshift. 
    The vertical gray lines denote FACS gates used. 
    Biological replicates are indicated by color. 
    \label{fig:figure4a}
    \textbf{(Bottom)}
    Distribution of mean modeled \textit{GAP1} mRNA levels for each mutant.
    The broad trends of changes in \textit{GAP1} abundance are
    captured, as well as some of the differences between replicates
    (replicate A is slightly lower than B in both timepoints, 
    replicate C had half as many sorted events and is 
    thus noiser in estimates).
    \label{fig:figure4c}
  }{Flow cytometry of the \textit{GAP1} mRNA distribution across all
    mutants, and global distribution of \textit{GAP1} mRNA abundance
    estimates.}

\afig{
  \includegraphics[width=.8\linewidth]{img/Figure4d.png}
  }{
    The mean \textit{GAP1} mRNA abundance estimated for 
    individual mutants before and after
    the upshift are shown as points connected by a line, colored
    according to the type of gene. 
    For reference, the background violin plot shows the distribution 
    of all 3,230 mutants fit.
    \label{fig:figure4d}
  }{Individual BFF estimates of \textit{GAP1} mRNA abundance for
    mutants known to play a role in \textit{GAP1} regulation.}

We found that the wildtype
genotype (\textit{his3}$\Delta$, complemented by a \textit{S. pombe} 
\textit{HIS5} gene in library construction) 
has a \textit{GAP1} abundance estimate that is centrally
located in the distribution both before and following the upshift. The
\textit{gap1}$\Delta$ genotype is a negative control and 
we estimate that it is at the extreme
low end of the distribution before and following the upshift. 
\textit{dal80}$\Delta$ is a direct transcriptional repressor
of NCR transcripts %like \textit{GAP1}, 
and we found that this strain is defective in
repression of \textit{GAP1} before and after the upshift. 
Counter-intuitively, deletion of \textit{GAT1}, a transcriptional activator
of \textit{GAP1}, appears to have higher steady-state abundance of
\textit{GAP1} mRNA.
However, increased abundance of \textit{GAP1} mRNA in a
\textit{gat1}$\Delta$ background has
previously been reported \parencite{scherens2006identification} and is thought to
result from the complex interplay of NCR transcription factors on
their own expression levels. 
Data and models for each mutant strain can be visualized in browser
using a Shiny application (see
\url{http://shiny.bio.nyu.edu/users/dhm267/} or \nameref{subsubsection:codeanddata}). 



\afig{
  \includegraphics[width=.49\textwidth]{img/Figure5_S_sulfateAssimilation.png}
  \includegraphics[width=.49\textwidth]{img/Figure5_S_negGluconeogenesis.png}
  }{
  Mutants deleted for genes involved in 
  sulfate assimilation \textbf{(left)} or
  negative regulation of gluconeogenesis \textbf{(right)} 
  are associated with higher \textit{GAP1} expresion 
  before \textbf{(left)} or after \textbf{(right)} the upshift, 
  by GSEA analysis of GO-terms (p-value < 0.05).
  \label{fig:gluco}
  \label{fig:sulfate}
  }{Mutants of negative regulators of gluconeogenesis or sulfate 
    assimilation are associated with defects in \textit{GAP1} 
    expression.}

To identify new cellular processes that regulate \textit{GAP1} mRNA abundance, we
used gene-set enrichment analysis (\nameref{itm:dme209gsea}).
Following the upshift we found that mutants with high \textit{GAP1}
abundance before the upshift are enriched for mutants in sulfate
assimilation, and mutants that 
maintain high \textit{GAP1} mRNA abundance are enriched for negative
regulation of gluconeogenesis (\autoref{fig:gluco}).
However, the strongest enrichment for high \textit{GAP1} abundance 
was components of the Lsm1-7p/Pat1p complex (\autoref{fig:figure5a}). 
Mutants in the TORC1 signalling pathway were not enriched,
but, I did find that a \textit{tco89}$\Delta$ mutant has
greatly increased \textit{GAP1} mRNA abundance before and after the 
upshift (\autoref{fig:tco89}), consistent with the repressive role 
of TORC1 on the NCR regulon.
To compare \textit{GAP1} abundance before and after the upshift for each mutant,
we regressed the post-upshift mean \textit{GAP1} abundance against the pre-upshift 
mean \textit{GAP1} abundance for each genotype (\autoref{fig:prePredictPost}). 
\afig{
  \includegraphics[width=.7\textwidth]{img/Figure4_S_poorlyQuantifiedStrains.png}
  }{
    Data and fits for several mutants. \textit{xrn1}$\Delta$ 
    mutant (left) is lowly abundant in
    the library and is only observed in the highest bin of \textit{GAP1} signal, consistent
    with the role of Xrn1p as a global exonuclease. 
    \textit{tco89}$\Delta$ is the only detected member that would abrogate TORC1 activity.
    This mutant (right) has elevated \textit{GAP1} mRNA before and after the upshift,
    consistent with the role of TORC1 in repressing the NCR regulon. 
    \label{fig:tco89}
  }{\textit{tco89}$\Delta$ and \textit{xrn1}$\Delta$ show
    defects in \textit{GAP1} mRNA regulation in the BFF assay.}
\afig{
  \includegraphics[width=.7\textwidth]{img/Figure4_S_PreShiftPredictingPostShiftLM.png}
  }{
    Scatter plot of the estimated \textit{GAP1} abundances, 
    with marginal histograms along top and right. 
    Blue vertical line on top histogram is a cut-off of
    \textit{GAP1} mRNA induction for this analysis,
    and is one standard deviation below the mean of the wild-type
    strain. The red linear regression line is fit to all points above this
    threshold, in which expression was detected before the upshift.
    \label{fig:prePredictPost}
  }{
    The pre-upshift mean has a significant but slight relationship 
    in predicting the post-upshift mean.
  }
We used the residuals for each
strain to identify mutants that clear \textit{GAP1} mRNA with kinetics slower
than expected by this model.
We found that the Lsm1-7p/Pat1p complex is again strongly 
enriched for slower than
expected \textit{GAP1} mRNA clearance (\nameref{itm:dme209pooledFits}). 
Specifically
the \textit{lsm1}$\Delta$, \textit{lsm6}$\Delta$, and 
\textit{pat1}$\Delta$ strains are highly elevated in \textit{GAP1}
abundance before the upshift and strongly impaired in the 
repression of \textit{GAP1} mRNA after the upshift
(\autoref{fig:figure5a}). 

\afig{
  \includegraphics[width=.9\textwidth]{img/Figure5a.png}
  }{
    In the background is the distribution of 
    fit \textit{GAP1} mRNA mean abundance levels for all mutants
    in the pool. Indicated by colored points and lines are the means for
    individual knockout strains, as labeled.
  \label{fig:figure5a}
  }{Disrupting the Lsm1-7p/Pat1p complex impairs
    clearance of \textit{GAP1} mRNA.}

\subsection{Testing the roles of decapping modulators and associated
components}

As these factors are associated with processing-body dynamics, 
we tested if microscopically-observable processing-bodies form or
disassociate during the upshift, using microscopy of Dcp2-GFP. 
We did not observe qualitative changes
in Dcp2-GFP distribution (\autoref{fig:pbodyScope}),
and thus the upshift does not
result in a microscopically visible changes in processing-body foci
as seen in other stresses. This is consistent with previous
investigations of amino-acid limitation stress
\parencite{hoyle2007stress} and
suggests that the defects in \textit{GAP1} mRNA clearance  
result from their roles in decapping or associated processes.

\afig{
  \includegraphics[width=\textwidth]{img/Figure5_S_pbodyMicroscopy.png}
  }{
  A strain harboring a copy of Dcp2p-GFP expressed from a plasmid
  was grown in conditions of exponential phase in YPD or 10 minutes of
  starvation in water (first row). Starvation in water is a common 
  condition known to result in the strong formation of processing-body 
  foci of Dcp2-GFP, and is thus a positive control.
  The bottom row shows microscopy during the upshift.
  We do not see either formation or dissolution of Dcp2-GFP foci 
  resembling p-bodies during the nitrogen upshift.
  \label{fig:pbodyScope}
  }{Processing-body dynamics are not associated with the
    nitrogen upshift, by Dcp2p-GFP microscopy.}

To confirm the role of the Lsm1-7p/Pat1p  complex in clearing \textit{GAP1}
mRNA during the nitrogen upshift we measured \textit{GAP1} mRNA
repression using qPCR normalized to
\textit{HTA1}, which is not subject to destabilization upon the upshift
(\autoref{fig:figure2a}). We also tested mutants that were not detected using BFF,
or were only detected in the highest \textit{GAP1} bin and therefore
not suitable for modeling
(e.g. \textit{xrn1}$\Delta$ \autoref{fig:tco89}). 
Using this assay we found that the main 5'-3' 
exonuclease \textit{xrn1}$\Delta$ 
and mRNA deadenylase complex (\textit{ccr4}$\Delta$ and
\textit{pop2}$\Delta$) are impaired in \textit{GAP1} repression 
(\autoref{fig:figure5qpcr}).
We found that qPCR confirms results from BFF.
We confirmed that the accelerated degradation of \textit{GAP1} mRNA is impaired
in \textit{lsm1}$\Delta$ and \textit{lsm6}$\Delta$ 
(\autoref{fig:figure5qpcr}). 
We also tested
\textit{scd6}$\Delta$ and \textit{edc3}$\Delta$, two modifiers of the
decapping or processing-body
assembly functions associated with this complex, and found two
distinct phenotypes (\autoref{fig:figure5qpcr}). \textit{edc3}$\Delta$
has similar abundance 
as wild-type before the upshift, but is cleared much more slowly.
\textit{scd6}$\Delta$ has a greatly reduced \textit{GAP1} abundance
before the upshift but is impaired in \textit{GAP1} clearance. 
\textit{tif4632}$\Delta$, a deletion of the eIF4G2
known to interact with Scd6p \parencite{rajyaguru2012scd6}, 
has a similar phenotype. 

\afig{
  \includegraphics[width=.49\textwidth]{img/Figure5b.png}
  \hfill
  \includegraphics[width=.49\textwidth]{img/Figure5c.png}
  \includegraphics[width=.49\textwidth]{img/Figure5d.png}
  \hfill
  \includegraphics[width=.49\textwidth]{img/Figure5_S_bothutr.png}
  }{
    Points are the ratio of \textit{GAP1} mRNA to \textit{HTA1} mRNA 
    before and 10 minutes after a glutamine upshift, in biological 
    triplicates. Lines are a log-linear regression fit. 
    Points are dodged horizontally for clarity, but this is not used
    for modeling. Wild-type is FY4.
    \textbf{(Top Left)} \textit{xrn1}$\Delta$, \textit{ccr4}$\Delta$,
    \textit{pop2}$\Delta$ are all slowed in clearance 
    (p-values < 0.004).
    \textbf{(Top Right)} \textit{lsm1}$\Delta$ and \textit{lsm6}$\Delta$ 
    are slowed in clearance (p-values < 0.0132 and 0.0299, 
    respectively).
    \textbf{(To Left)} \textit{edc3}$\Delta$ is slowed in clearance 
    (p-value < $10^{-4}$). \textit{scd6}$\Delta$ and 
    \textit{tif4632}$\Delta$ are slowed in clearance 
    (p-values < $10^{-5}$) and have lower mRNA abundance
    before the upshift (p-values < 0.003).
    \textbf{(Top Right)} A deletion of 150bp immediately downstream 
    of \textit{GAP1} stop 
    codon has no significant effect, but a deletion of 100bp
    immediately upstream of the start codon has slower clearance 
    (p-value < $10^{-4}$) and 
    lower level of \textit{GAP1}/\textit{HTA1} before the upshift (p-value < 0.0015).
    During strain construction, a deletion of 152bp 5' of the start
    codon was also generated, and expresses a similar phenotype 
    ( p-value < $10^{-5}$ clearance defect, p-value < $0.0061$ ).
    \label{fig:figure5qpcr}
  }{Disrupting core pathways of mRNA degradation, decapping
    modulators, or the 5' UTR impairs the clearance of \textit{GAP1} 
    mRNA, by qPCR}

Identification of an initiation factor subunit with defects in
\textit{GAP1} mRNA clearance suggests that translation control may
impact stability changes. Therefore we deleted sequence of 
the 5' UTR and 3' UTR of \textit{GAP1}, specifically 100bp and 152bp 
upstream of the start codon (approximate 5' UTR) or the
100bp downstream of the stop codon (approximate 3' UTR).
Whereas the 3' UTR deletion does not have an effect, the 5' UTR deletion
exhibits the phenotype of reduced \textit{GAP1} mRNA before the upshift
and a reduced rate of transcript clearance following the upshift
(\autoref{fig:figure5qpcr}). 
We observed a similar phenotype with a different deletion of 152bp upstream
of the \textit{GAP1} start codon (\autoref{fig:figure5qpcr}). 
%These measurements suggest altered mRNP composition of the
%Lsm1-7p/Pat1p complex and associated decapping factors are associated
%with defects in \textit{GAP1} mRNA  dynamics upon a 
%nitrogen upshift. Importantly the phenotype of the \textit{scd6}$\Delta$, 
%\textit{tif4632}$\Delta$, or 5' sequence deletions preceed the 
%addition of glutamine, suggesting that the observed destabilization
%of \textit{GAP1} may be the halt of a stabilization effect, perhaps
%due to changes in translational status of \textit{GAP1}.
This indicates that \textit{cis}-elements responsible for the
rapid clearance of \textit{GAP1} are unlikely to be located in the
3' UTR, and instead may be exerting an effect at the 5' end of the
mRNA.

\subsection{Methods and materials}

\subsubsection{Strains}

Strains with deletions 5' of the start codon and 3' of the stop
codon were generated by the "delitto-perfeto" 
method \parencite{storici2006delitto}, 
by inserting the pCORE-Kp53 casette
at either the 5' or 3' end of the coding sequence, then transforming
with a short oligo product spanning the deletion junction and
counter-selecting against the casette with Gal induction of p53 
from within the cassette.
These strains were generated and confirmed by Sanger sequencing,
and traces are available in directory \texttt{data/qPCRfollowup/} 
within the data zip archive (\nameref{subsubsection:codeanddata}).

\subsubsection{qPCR}

Each strain was grown from single colonies.
Samples were collected before, during the first ten minutes of
the nitrogen upshift (\autoref{fig:figure3a}),
or at ten minutes after the upshift (\autoref{fig:figure5qpcr}).
For the experiments described in
\autoref{fig:figure5qpcr}, all work
was done in biological replicates.
Each 10mL sample was collected by vacuum filtration, and RNA extracted
using the hot-acid phenol technique.
For \autoref{fig:figure3a} only, at the beginning of this extraction 
incubation I added 10$\mu$L of a 0.1ng/$\mu$L in-vitro synthesized 
spike-in mRNA BAC1200 (as generated
for the label-chase RNAseq (\nameref{subsubsection:spikeins},
but without 4-thiouridine). 
RNA was treated with DNAse RQ1 (Promega) according to manufacturer
instructions, cleaned and precipitated.
All samples were reverse transcribed. 
For \autoref{fig:figure3a} 2$\mu$g RNA was primed with 2.08ng/$\mu$L
random hexamers (Invitrogen) and 2.5mM total dNTPs (Promega),
while for \autoref{fig:figure5qpcr} 1$\mu$g RNA was primed with 
5.6mM Oligo(dT)18 primers (Fermentas) and
0.56mM total dNTPs (Promega).
These mixtures were reverse transcribed with M-MulvRT (NEB) according
to manufacturer's instructions, then diluted 1/8 with hyclone water 
and used as direct template in 10$\mu$L reactions with SybrGreen I 
Roche qPCR master-mix (Roche). These were measured
on a Roche Lightcycler 480. 
For \autoref{fig:figure3a}, I used primers 
DGO230,DGO232 to quantify \textit{GAP1} and 
DGO605,DGO606 to quantify the synthetic spike-in BAC1200.
For \autoref{fig:figure5qpcr}, Nathan Brandt used primers
DGO229, DGO231 to quantify \textit{GAP1} and
DGO233, DGO236 to quantify \textit{HTA1}.  
See \autoref{tab:primerTable} for sequences.
These were run on a Roche480 Lightcycler, 
with a max-second derivative estimate
of the cycles-threshold (the $C_p$ value output by analysis) used 
for analysis by scripts included in the git repo 
(\nameref{subsubsection:codeanddata}).
Linear regression of the log-transformed values was used to quantify
the dynamics and assess significance of changes in mRNA abundance
levels or rates of change.

\subsubsection{Microscopy of Dcp2-GFP}

To look for processing-body dynamics in response to
a nitrogen upshift, I used strain DGY525, which is FY3
containing plasmid pRP1315 (gift from Roy Parker).
Samples were collected before and following a nitrogen upshift
(4, 10, 12, 19, or 25 minutes later),
from exponential growth in YPD, 
or 10 minutes after resuspending YPD-grown cells in DI water.
All samples were collected by brief centrifugation (1 minute)
then resuspension in PBS buffered 4\% PFA for 
aspirating most supernatant, then centrifugation for 20 seconds
and aspirating all media. Each pellet was 
immediately resuspended in 4\% PFA 
(diluted from EMS 16\% PFA ampule RT15710) 
with 1x PBS for 15 minutes on bench, 
then spun at 10,000g for 1 minute, aspirated, 
then washed once and resuspended with 1x PBS. 
Samples were kept on ice, then put onto a coverslip
for imaging on a DeltaVision scope. Raw images available in the
microscopy zip archive (\nameref{subsubsection:codeanddata}).

\subsection{Methods and materials of Barseq after FACS after FISH experiment}

\subsubsection{Culturing and sampling}

An aliquot of the prototrophic deletion collection
\parencite{vandersluis2014broad} was thawed and diluted, with 
approximately 78 million cells added to 500mL of proline-limited media 
in a 1L baffled flask. This was shaken at 30$^{\circ}$C overnight, 
then split into three flasks (A, B, and C). 
After three hours (at mid-exponential)
we collected samples of 30mL culture by filtration and flash-freezing.
The collection times for each
sample were A: 10 minutes and 38 seconds, B: 10 minutes and 12 seconds
C: 10 minutes and 17 seconds.

\subsubsection{Fixation and permeabilization}

Frozen samples of the pool were fixed with formaldehyde
(4\% PFA diluted in PBS, 2 hours room-temperature) and digested
with lyticase (in BufferB with VRC 37C 1 hour).
Microscopy monitoring of the reaction showed the classic
greying of the cells under phase contrast microscopy to a dark grey, 
but did not digest to ghosts and fragments. 
Critically, 1/5th volume 2.5M glycine was added to quench the 
fixation before pelleting and washing the cells with centrifugation
at 1200g room temperature. An experiment where quenching did not
take place resulted in smaller fixed cells that were less accessible
to the FISH hybridization.
After washing three times with Buffer B \parencite{pringle199140} 
these were permeabilized with ethanol at
4C overnight.

\subsubsection{Hybridization}

The samples were processed with a Quantigene Flow RNA kit purchased in
March of 2015 (product code 15710), and designed for GAP1 mRNA in
\emph{Saccharomyces cerevisiae}. The probe sequences are proprietary.
This procedure is largely as described by the manufacturer, with some
critical modifications.
The incubator used was calibrated to 40\(^{\circ}\)C using a Traceable
4004 Type-K thermometer, with the probe inserted into an eppendorf tube
through a hole and sealed with parafilm, and inserted into the aluminum
heatblock in the air incubator as used for incubating samples. 
The ethanol-permeabilized samples were pelleted by centrifuging 
1200g for 5 minutes room temperature, then washed with "Solution D"
from the kit before proceeding with the kit instructions, with the
exception that all reactions were conducted with 1/4 volume. 
For each wash, the complete supernatant was discarded, and cells
resuspended in wash buffer to 25$\mu$L (to replicate the "about
100$\mu$L residual" in the instructions).
Upon final wash, these were incubated for 5 minutes with DAPI
to counter-stain.

\subsubsection{Flow cytometry and FACS}

Samples were sonicated, then run through a BD FACSAria II
by a NYU GenCore technician.
Cells were gated for singlets and DAPI content 
(estimated 1N or more), then sorted based on emission area from a
660/20nm filter with a 633nm laser activation
into four gates within each timepoint, across replicates.
Importantly, the sorting gates were set with a GUI interface until they
approximated splitting the libraries into quartiles for the six samples.
These were sorted using PBS sheath fluid at room-temperature, into
poly-propylene FACS tubes, then stored at -20$^{\circ}$C.

Note that we later in analysis add a fixed number to all observations
and gates in linear scale in order to get into positive values.

\subsubsection{Cell collection and DNA extraction}

For each gate, cells were collected in eppendorfs via laborious 
repeated gentle centrifugation steps and genomic
DNA extracted by NaCl reverse-crosslinking at 65$^{\circ}$ for 16
hours, inspired by \cite{klemm2014transcriptional}, with
subsequent proteinase K and RNase A digestions.
DNA was extracted from each sample with a very careful
phenol:chloroform extraction, with back-extraction, estimated to yield
1/3rd the theoretical input (by qubit and estimation), twice as good
as by either Zymo columns or Ampure beads. This may be a result of
trying to obtain genomic DNA.

\subsubsection{Construction of amplicon sequencing libraries for barcode counting}

gDNA was amplified in technical triplicates in a heavily modified 
BarSeq protocol.

A master mix of buffer, BSA, dNTPs, MgSO$_4$, and Vent (exo-)
polymerase (NEB) is made containing three oligos DGO1562, DGO1588, 
and DGO1589.
This is combined with about 1/3rd of the genomic DNA extracted
from the sample, and the reaction is loaded into individual PCR
tubes (the optimization was in individual PCR tubes and 96 well 
plates did not work, likely due to particularities of the sealing
mechanism).
30 reactions were run in each batch of preparation. All 30 reactions
were put into a BioRad T100 thermocycler set for a 30\(\mu\)L
reaction.

This was cycled through 4 minutes at 95C to denature, then a single
annealing and extension step. This was then cooled to 37C and
immediately ExoI (Thermo) was added and incubated (with periodic
mixing) for twenty minutes. This exonuclease proceeds from
un-annealed 3' ends. 


The exonuclease was then inactivated with a 80C incubation, 5 minutes,
then DGO1576, DGO1567, and DGO1519 were added in a master mix
containing glycerol to bring the whole reaction to about 5\% glycerol
content. This became essential for keeping the forward primer
effective at lower primer concentrations, presumably due to the
computationally predicted hairpin in the 3' end.

This reaction was cycled 40 times to anneal and extend the product.
I do not expect all the cycles to be productive. 
Rather, the lower concentrations used
(10nM forward DGO1567) limit the product generation.

To this generated intermediate product, I then added more primers,
more buffer, and indexed forward primers. These incorporate a 5bp
sample index. This is of insufficient complexity for error-correction
bioinformatically, but should be sufficient to demultiplex most
samples. This was cycled 12 times to extend the product.

All reactions were then frozen, then thawed on ice and pooled into
four pools. These were purified, then one last reaction used the
same polymerase to do three rounds of extension to add on the final
Illumina P5 adapter onto the molecule. This was resolved on a 3\%
agarose gel, then the only visible bands were purified, quantified
using qPCR, and submitted for sequencing on an Illumina NextSeq.

Based on previous runs, the Genomics Core spiked-in 5\% PhiX onto
the run, to help maintain diversity. However, during the run there was
a massive failure in Illumina software to discern clusters given
low base-diversity.  This run only yielded approximately 127 million
reads (out of 400 million listed yield). Thus, 5\% PhiX is too low,
and I would recommend trying $\sim$25\% in future runs. 

\subsection{Design and analysis of Barseq after FACS after FISH experiment}

\subsubsection{Design of Barseq after FACS after FISH experiment}

\label{subsection:bff}

First, we motivate this development, as it departs from previous procedures
in a few ways.  
The main impetus for this was the generation of primer dimers that form
when the forward universal primer primes off the reverse universal
primer. For example, \autoref{fig:dimer} shows a failed experiment
that shows dimer formation in the sample lanes (on right). 

\vspace{3em}

\afig{
  \includegraphics[width=.5\textwidth]{img/exampleDimers.png}
  }{
\begin{tikzpicture}[overlay
    ,font=\small,%font=\ttfamily
    ,inner sep=0pt,outer sep=0pt
    ,shift={(0.7,-3)}]
    ]
  \node[align=left] at (-5.8,5) (ladder200) {200bp};
  \node[align=left,below=0.6cm of ladder200] (ladder100) {100bp};
%
  \node[below right=0.025cm and 1cm of ladder200] (200L) {};
  \draw[->] (ladder200) -- (200L);
  \node[above right=0.000cm and 1cm of ladder100] (100L) {};
  \draw[->] (ladder100) -- (100L);
%
  \node[align=left,anchor=south] at (-3.5,5.6) {Ladder\\Lane 1};
  \node[align=left,anchor=south] at (-2.1,5.6) {Lane 2};
  \node[align=left,anchor=south] at (-0.7,5.6) {Lane 3};
  \node[align=left,anchor=south] at (0.7,5.6) {Lane 4};
  \node[align=left,anchor=south] at (2.1,5.6) {Lane 5};
  \node[align=left,anchor=south] at (3.4,5.6) {Lane 6};
\end{tikzpicture}
    Here, I show an example of the primer-dimer formed by indexed
    versions of the universal deletion-collection barcode primers. 
    Products after presumable addition of Illumina P5 and P7 adapter
    sequences were run on a 3\% TAE agarose gel and 
    stained with Sybr Safe dye.
    The left-most lane is a NEB 100bp ladder, with the bottom 
    two bands as 100bp and 200bp. The red is due to overexposure.
    The right five bands are from samples prepared with an earlier
    version of this protocol. The band approximately 190bp is
    throught to be the library product, and the band approximately
    160bp is thought to be the inhibitory and unwanted 
    barcode-less dimer. Lane 3 clearly shows both bands, while lane
    6 is all dimer.
  \label{fig:dimer}
  }{An example of a dimer.}

Below a critical threshold, this dimer greatly out-competes the desired
product and can result in a loss of amplicon before the amplicon is
amplified enough to gel extract (above figure, lane 6). The dimer is
also sequenced via Illumina chemistry (not desired). By Sanger
sequencing we found that it appears to result from a three base
truncation of the forward primer priming perfectly for about 6 bases off
the reverse primer. This was not solved by switching to a polymerase
without 3' exonuclease activity, or by using HPLC purified primers.
Using different reverse primers lead to off-target products.

We saw these dimers before incorporating a UMI step into the protocol.
We used a UMI because we wanted the protocol to be as quantitative as
possible, despite the multiple amplification steps that would introduce
randomly sampled noise at each cycle. The design of this was 6 bp
degenerate sequence spaced with fixed bases, in the design of
\texttt{NCNCNCNTNCN} because we estimated this would best block
annealing to any 3' ends of the primers used. In future work, we would
strongly recommend using more degenerate bases for such a low-complexity
library \parencite{fu2011counting}. 

In order to digest the
excess un-incorporated UMI primers, addition of a
exonuclease is required. 
ExoI is characterized to be maximally effective at
37\(^{\circ}\)C, and although it can have activity at 42\(^{\circ}\)C
for some time \parencite{fei2015structural}, it
will be inactivated. This low temperature requirement likely exacerbates
dimer and off-target product formation.
The common, but incorrect, one-UMI-one-molecule assumption is based on
the belief that each UMI and each genomic template is used once in the
first round. The exonuclease step must therefore digest all
un-extended UMI-containing primers for this to be true. One control
for this is to monitor the primer concentrations using a technology
like HPLC to measure abundance, but a small number is, again, not
necessarily negligible. The proper control is to not use the
polymerase to extend the first round, and instead immediately digest
the UMI-containing primers. This is a crucial control, and on the
basis of experiments I did using this setup using dilutions of
template, I estimate that the effect of the exonuclease, in the 
absence of the first round of extension, reduced product formation 
by about 20 fold. However, it was not absent. Thus, I suspect that 
the exonuclease digestion only ensures that $\approx$20/21 UMIs are
from the first round of incorporation. While small, this is not
negligible, especially in conditions where template number changes
dramatically (as here it does not, given similar inputs to extractions).
Optimization of this step did not improve upon this efficacy, and in
the lack of commercially available thermostable (and active) 
exonucleases \parencite{fei2015structural}, this background noise 
must be understood and accepted.

To address this, we optimized the reactions on a dilution series of gDNA
from a different experiment with the same knockout library. By balancing
MgSO\(_4\) and glycerol concentrations, we got better amplification and
then tried to use a ``booster'' \parencite{ruano1989biphasic} PCR approach. 
This gave some improvements in how low we
could detect before saturating the reaction with dimers, but we could
not go lower in primer concentration and attributed this to the
predicted secondary structure in the 3' end of the primer amplifying
from the outside of the UPTAG barcodes. Adding DMSO helped with this,
but we still had to leave the reaction with plenty of primer as
intra-molecular interference from this process would out-compete
inter-molecular productive annealing. We still could not get reliable
amplification from \textless{} \(10^5\) templates (estimated by qubit
assuming 12.5 picograms gDNA per genome).

The major solution to this problem was the addition of 3' phosphorylated
blocker oligos. These are not extended by DNA polymerases but are
displaced by a strand-displacing polymerase like Vent exo-. By using
this polymerase and blocker combination, we prevent new 3' ends from annealing
but allow properly annealed primers to extend through this region. This,
in combination with the exonuclease digestion of most of the reverse
primer, prevented dimer formation. This revealed that these universal
priming sequences will amplify from two loci near \emph{CIA1} and
\emph{RDN37}. This was identified by Sanger sequencing gel-extracted
bands, so we designed more 20-mers that again block off-target annealing
and found they worked wonderfully. In test experiments, we believe we
got amplicons of the correct size from as low as \(\sim\) 300 targets
but have not sequence verified this.

To simplify the addition of the last 5' Illumina P5 adpater, we kept
this as a separate reaction. To minimize chimera formation between
different samples in this reaction, this is a 2-step polymerase
extension reaction which partially forms the sequencing product (1/3 of
results, theoretically). This is sufficient for qPCR quantification of
the library and Illumina sequencing. Given our gel-extraction clean up
and small product size, we do not expect formation of chimeras on the
flow cell.
%\footnote{http://dnatech.genomecenter.ucdavis.edu/2017/04/11/update-on-barcode-mis-assignment-issue/}

\autoref{fig:cartoon} shows a cartoon of
the amplicon library-making procedure, up to the generation of the
product before the Illumina P5 addition \autoref{fig:amplicon}. 
Given the simplicity of this
last reaction (simply adding sequence on the 5' end), it is omitted.

\afig{
  \vspace{-4em}
  \includegraphics[height=10.0in]{img/sobaseqPCRschematic-1.png}
  \vspace{-7em}
  }{
  \label{fig:cartoon}
  }{A schematic of the barcode sequencing strategy.}

%  \raggedright
%  \begin{minipage}{1cm}
%    \caption{}
%  \end{minipage}
%  \includepdf[pages={2},scale=0.8]{img/sobaseqPCRschematic.pdf}


\afig{
\begin{tikzpicture}[overlay
    ,font=\small,font=\ttfamily
    ,inner sep=0pt,outer sep=0pt
    ,scale=0.8, every node/.style={scale=0.7}
    ,shift={(-9.25,-3)}
    ]
  \node[fill=orange!50,align=left,anchor=west] at (-1,0) (1) 
    {\scriptsize ACGCTCTTCCGATCT};
  \node[fill=green!30,align=left,anchor=west] (2) at (1.east) 
    {\scriptsize NNNNN};
  \node[fill=yellow!100,align=left,anchor=west] (3) at (2.east) 
    {\scriptsize GTCCACGAGGTCTCT};
  \node[fill=white,align=left,anchor=west] (4) at (3.east) 
    {\scriptsize NNNNNNNNNNNNNNNNNNNN};
  \node[fill=yellow!100,align=left,anchor=west] (5) at (4.east)
    {\scriptsize CGTACGCTGCAGGTCGAC};
  \node[fill=purple!30,align=left,anchor=west] (6) at (5.east)
    {\scriptsize NGNANGNGNGN};
  \node[fill=orange!50,align=left,anchor=west] (7) at (6.east)
    {\scriptsize GATGTGACTGGAGTTCAGAC};
  \node[fill=cyan!50,align=left,anchor=west] (8) at (7.east)
    {\scriptsize ATCTCGTATGCCGTCTTCTGCTTG};
%
  \node[align=left,below=0.5cm of 1] (1l) {adaptor\\sequence};
  \node[align=left,above=0.5cm of 2] (2l) {sample\\index\\sequence};
  \node[align=left,below=0.5cm of 3] (3l) {fixed\\sequence};
  \node[align=left,above=0.5cm of 4] (4l) {strain barcode};
  \node[align=left,below=0.5cm of 5] (5l) {fixed\\sequence};
  \node[align=left,above=0.5cm of 6] (6l) {UMI};
  \node[align=left,below=0.5cm of 7] (7l) {adaptor\\sequence};
  \node[align=left,below=0.5cm of 8] (8l) {illumina\\adaptor sequence};
%
  \draw[->] (1l) -- (1);
  \draw[->] (2l) -- (2);
  \draw[->] (3l) -- (3);
  \draw[->] (4l) -- (4);
  \draw[->] (5l) -- (5);
  \draw[->] (6l) -- (6);
  \draw[->] (7l) -- (7);
  \draw[->] (8l) -- (8);
\end{tikzpicture}
\\\vspace{9em}
  }{
  \label{fig:amplicon}
  }{The expected amplicon, before adding P5 sequences at the 5' end.}

These were checked with sanger sequencing, for pools 1 and 3, using
primers DGO 276 or DGO 1519, with Genewiz sequencing. Representative
Sanger sequencing image of library pool 1, sequenced forward (with DGO
216) is shown in figure \ref{fig:sanger}.

Trace colors: red is T, green is A, blue is C, black is G.

\afig{
\includegraphics[trim={0cm 1.6cm 0cm 3cm},clip,width=\textwidth]
  {img/represtativeSangerImage.png}
\begin{tikzpicture}[overlay,shift={(-8.25,0)}]
  \node[rotate=90,anchor=west] at (3,5) (sample) {5bp sample barcode};
  \draw[->] (sample) to (2.7,3.5);
  \draw[->] (sample) to (3.3,3.5);
  \node[rotate=0,anchor=west,align=left] at (6,5) (strain) {15-22bp \\strain barcode};
  \draw[->,bend right=20] (strain) to (6,2.5);
  \draw[->,bend left=20] (strain) to (9,2.5);
  \node[rotate=0,align=left,anchor=west] at (10,7) (umi) 
    {UMI barcode, 5bp fixed \\and 6bp degenerate};
  \draw[->,bend left=00] (umi) to (12.20,2.5);
  \draw[->,bend left=04] (umi) to (12.50,2.5);
  \draw[->,bend left=08] (umi) to (12.80,2.5);
  \draw[->,bend left=12] (umi) to (13.10,2.5);
  \draw[->,bend left=16] (umi) to (13.40,2.5);
  \draw[->,bend left=20] (umi) to (13.75,2.5);
\end{tikzpicture}
  }{\label{fig:sanger}}
  {Sanger sequencing of the produced library shows the expected
  degenerate positions.}

\subsubsection{Designing an analysis pipeline to filter dimers and 
extract UMIs from indeterminate locations}

We previously used the one-program solution of BarNone 
(\url{http://varianceexplained.org/BarNone/})
to rapidly and easily quantify barcode counts from yeast barcode
sequencing experiments. However, our new amplicon design makes use of
UMIs to help account for the amplicon noise inherent in the BarSeq
method, and BarNone does not account for these. We also wanted to devise
a pipeline that would be modular, consisting of multiple well-designed
tools that could be modified independently and would maintain read
information along the pipeline to assist in debugging and benchmarking.

\begin{enumerate}
  \setlength\itemsep{1em}
  \item
  FASTQ files of the reads are fed into a custom python script called
  \texttt{SLAPCHOP.py}. This is named because it Simply Looks At
  Pair-wise Comparisons to Help Optimize Parsing 
  (\url{https://github.com/darachm/slapchop}). This
  parallelized script takes each read, aligns the expected fixed
  sequences that bracket the informative barcodes (using
  BioPython \cite{cock2009biopython}), decides if the read matches the
  expected structure based on a specified criteria, then extracts out 
  the sample index, strain barcode, and UMI degenerate sequence into 
  appropriate positions in a FASTQ format. This keeps the filtering 
  and strain barcode identification separate from the fixed sequences.
  \item
  A simple perl script (\texttt{pickyDemuxer.pl}) demultiplexes the
  processed FASTQ file on perfect matches of the 5bp index sequence and
  generates a demultiplexing report.
  \item
  The strain barcode regions, padded with flanking sequence to a uniform
  length of 26bp, from the demultiplexed FASTQ files are aligned using
  \texttt{bwa\ mem} \parencite{li2013aligning}
  to the
  expected barcodes as re-annotated by \cite{smith2009quantitative}.
  \item
  From the resulting \texttt{bam} alignment files, we extract the strain
  identification and UMI. Using the UMI-collision / label-saturation
  concept and equation of Fu et. al. 2011 (PNAS), we adjust the
  saturated pool to estimate the input of strain genomes into the
  library preparation.
\end{enumerate}

This allowed us to recognize and extract barcodes from indeterminate
positions in the amplicon and filter the reads for real, intact
amplicons in one step. This extraction improved our accuracy, for
example eliminating spurious alignments of the forward priming sequence
against the barcode of \emph{ymr258c}\(\Delta\), a barcode re-annotated
with striking similarity to the fixed priming sequence
\parencite{smith2009quantitative}.

There are several different ways to use the UMI information to estimate
unique input molecules from a sequencing assay. The naive approach is to
assume that every combination of strain barcode with a certain UMI
sequence in a sample is a unique event, and any repetitions of this are
only PCR duplicates. 
The molecular biology caveats for this assumption are addressed 
earlier.
However, we only have \(4^6=4096\) possible UMIs for \(\sim4500\) 
possible strain
barcodes, with approximately a million reads per sample. Thus, due to a
short length of UMI we cannot make this assumption, and instead are
confronted with a space of UMIs with a fairly high chance of two
UMI-strain combinations being generated by random chance alone. 

We refer
to this as a UMI-collision (similar concept to a hash collision) or the
phenomenon of label saturation. 
There are multiple ways to deal with this.
The simplest is to simple take each unique UMI, but errors in library
amplification could diversify spuriously duplicated UMIs.
Error-correcting algorithms 
exist that use graph information to improve accuracy of this method, but
these require that the space of all usable UMIs is sparse. 
Another solution is the label saturation correction of 
\cite{fu2011counting}. This depends
on treating the chance that any UMI-collisions are a random and rare
event, thus modeled as a Poisson distribution, similar to the classic
Luria-Delbruck method of mutation rate estimation. 
If we have 4096 possible
UMIs, and for one strain in one sample we observe \(x\) different UMIs
associated, then we estimate that there were \(z\) different original
molecules in the sample, where
\(z=4096 (1-e^{-\frac{x}{4096}})\) .

\label{subsubsection:rarefractionDiscussion}

\autoref{fig:umi} plots this function, and compares it to
actual sequencing data from the experiment, with and without
the error-correcting approach of UMI-tools \parencite{smith2017umi}.
\autoref{fig:umihistz} plots the distribution of counts per UMI
for one mutant (the most abundant one in the library). 

\afig{
  \includegraphics[width=.49\textwidth]{img/Figure4_S_umiSaturationCurve.png}
  \includegraphics[width=.49\textwidth]{img/Figure4_S_totalInputReadsVsUMItoolsCounts.png}
  }{
  For each sample, each combination of UMIs and strain barcodes was
  collected. For each point, the y-axis denotes the unique UMIs
  observed for that combination of strain and sample.
  On the x-axis is the raw counts of observing that strain barcode
  in that sample, without error-correction
  \textbf{(Left)} and with \texttt{umi-tools} error-correction
  algorithim on the right \textbf{(Right)}.
  The line denotes the curve expected just from label saturation
  with increased re-sampling of a limited pool as it approaches 4096. 
  \label{fig:umi}
  }{A comparison of the unique UMIs versus input UMIs for un-corrected
    and error-corrected UMIs.}

\afig{
  \includegraphics[width=.8\textwidth]{img/umi_histograms.png}
  }{
  In black is the observed data, in red is the expected distribution
  if Poisson.
  We see that there is a long-tail of UMIs with more reads than 
  observed for that combination of strain and sample.
  \label{fig:umihistz}
  }{Histograms of UMI observations associated with the YOR202W barcode
    in three samples.}

Comparing these, we see that the unique UMIs detected decreases as a
function of input-reads, as expected. 
We see that the error-correction algorithim greatly reduces the reads
used, and falls well below this expectation. As shown in
\autoref{fig:umihistz}, we see that for some UMI-strain barcode
combinations, we see much more counts per UMI label than expected by a
Poisson distribution. 
Both label saturation and PCR duplication
are at play to distort the mapping between either raw counts or unique
labels and the actual underlying estimate of input genome targets per
strain in each sample.
However, since UMI-tools uses abundance of counts per UMI-cluster
to aggregate neighboring UMIs into the cluster, I believe that the
PCR-duplicate UMIs that are overdispersed on the high-counts-side
are scavenging the lower end of the poisson distribution 
into a small number of real clusters. This is not
surprising given that we are using this tool in a labeling regime that
is wholy inappropriate for what it is trying to do. 
Thus, I do not use
this tool and instead used the more conservative 
correction of \cite{fu2011counting}, \autoref{fig:umi} left.
An appropriately complex UMI design would allow a future user to make
use of error-correcting algorithims in this method.

\subsubsection{Benchmarking this pipeline}

We benchmarked this pipeline against an \emph{in silico} dataset to
determine performance across a range of mutation rates and read depths
similar to what we would expect for this experiment. We also compared
the two UMI correction approaches described in the previous section, the
``UMI-collision correction'' and the ``number unique'' method. We used a
python script \texttt{makingFakeReads.py} to generate several datasets
with the following parameters:

\begin{itemize}
  \setlength\itemsep{0em}
  \item
  16 million reads per FASTQ, split amongst 32 samples
  \item
  each strain barcode is sampled from an emipirically observed
  distribution averaged from the first timepoints of an unpublished
  dataset, quantified by BarNone
  \item
  each amplicon has a poisson number of random single nucleotide
  mutations to a different base, based on a given parameter of
  \texttt{0}, \texttt{1}, \texttt{2}, or \texttt{3} lambda of mutations
  per amplicon.
  \item
  each generated amplicon is added to the file \(x\) number of times,
  where \(x\) is an exponential distribution with mean 5
  \item
  3 ``biological'' replicate datasets are generated per set of
  parameters
\end{itemize}

After quantification, we calculate pearson correlation, spearman
correlation, and the number of mistaken strain identifications.
Tolerated mismatches is a parameter set in BarNone or by the score
requirements for alignment in \texttt{bwa}.

As seen in \autoref{fig:wholeSampleDme234}, 
it would appear that by pearson correlation, the filtration step of the
\texttt{bwa} alignment allows us to make more robust assignment of
strain barcodes. The spearman correlation tells us that as mutation rate
increases, high mismatch tolerance on the \texttt{bwa} tool is very
dangerous for misaligning and can cause large rank changes.

\afig{
  \includegraphics[width=\textwidth]{img/fig4s_whole_sample_pearson.png}
  \includegraphics[width=\textwidth]{img/fig4s_whole_sample_spearman.png}
  }{
  Tool quantification was compared to the \textit{in silico} ground
  truth using correlations of Pearson \textbf{(top)} or Spearman
  \textbf{(bottom)}.
  \label{fig:wholeSampleDme234}
  }{Optimization of mutant quantification methods, looking at
    simulated data on a whole-lane comparison.}

How does the UMI-collision correction perform?
We see that on the whole, un-demultiplexed datasets (16 million reads
across \textasciitilde{}4000 strains and 4096 possible UMIs) that the
performance is best with the UMI-collision correction 
\autoref{fig:dedupSampleDme234whole}. We see that just
using unique UMI counts in this regime leads to a good reconstruction of
the rank order (Spearman), but inaccurate of the magnitude (Pearson)
against ground truth.
Does this change with lower read density? 
Each point is one of 32 demultiplexed samples of three whole-library
replicates, and we see that in this lower read regime, we get similar
performance from both methods \autoref{fig:dedupSampleDme234per}.
To avoid biases arising from differences in abundance between strains,
I keep the UMI-collision correction.

\afig{
  \includegraphics[width=\textwidth]{img/fig4s_whole_sample_dedup_pearson.png}
  }{
  On a whole-sample basis, the error of the "number unique" quantifier
  of UMIs is evident, in this saturated regime.
  \label{fig:dedupSampleDme234whole}
  }{Optimization of mutant quantification methods, looking at
    deduplicating methods on whole comparisons.}

\afig{
  \includegraphics[width=\textwidth]{img/fig4s_demux_dedup_pearson.png}
  }{
  On a per-multiplexed-sample basis with less total counts, 
  the error of the "number unique" quantifier
  of UMIs is less evident.
  \label{fig:dedupSampleDme234per}
  }{Optimization of mutant quantification methods, looking at
    deduplicating methods on per-sample comparisons.}

BarNone appears to be more robust to mutations, in that it maintains a
flatter profile in the higher regimes of mutations. However,
\texttt{bwa} starts out higher. Accounting for duplication, as generated
by the exponential distribution described above, greatly improves
performance. At an average coverage here of a half-million reads, the
difference between the UMI-collision and unique counts is less than
with large coverage.  
In conclusion, our pipline for analysis is able to use an early
filtration step (SLAPCHOP) to improve strain barcode
identification and to extract UMIs that are useful for de-duplicating
PCR duplicates for better estimates of the ground truth.

\subsubsection{Modeling \textit{GAP1} FISH signal per strain in the pool}

In order to use the counts of each mutant in each sample to estimate
\emph{GAP1} mRNA abundance per strain, we used a maximum-likelihood
modeling approach.

We are interested in the number of cells of a certain strain that went
into each bin. We estimate this as a metric we define as
``pseudocounts'', or \(u_{ik}\) where \(i\) is the strain index, and
\(k\) is the FACS bin. We call the sequencing counts \(c_{ijk}\), where
\(j\) is the particular PCR replicate out of \(J\) PCR replicates that
were successfully sequenced. We assume the sequencing counts are
linearly amplified from the ``events'' of actual cells being sorted into
each collection tube, and we assume that all of these ``events'' have
equal chance to be amplified and detected by this sequencing assay. Then
we scale this estimate by the total number of ``events'' we observed
during the FACS procedure going into each bin \(e_k\). We assume that
all ``events'' had equal chances in all bins to get sequenced. Then we
have

\[ u_{ik} = 
  \frac{ \sum_j \frac{c_{ijk}}{\sum_i c_{ijk}} }{J}
  \;\;\; \frac{ e_{k}}{\sum_{k} e_{k}} \]

This is intuitively more simple than the notation used here to describe
it precisely. Since we split the library into quartiles for the
sequencing, \(\frac{ e_{k}}{\sum_{k} e_{k}}\) is about one quarter for
each bin. \(\frac{ c_{ijk}}{\sum_{i} c_{ijk}}\) is just the proportion
of counts in that sample that are that mutant \(i\).
\(\frac{ \sum_j \frac{c_{ijk}}{\sum_i c_{ijk}} }{J}\) is simply the
average of the proportion of counts, across the PCR replicates.

Thus, \(u_{ik}\) is essentially the proportion of the
original library that is the mutant in that bin, and 
\(\sum_k u_{ik}\) is the total proportion of the library that is
that mutant.
So then if we divide \(\frac{u_{ik}}{\sum_k u_{ik}}\),
we have an estimate of the proportion of that mutant that
went into each bin, out of all the mutant that was in the experiment.

Once we have this normalized pseudocounts metric within each biological
replicate, then we fit a log-normal model. We explored a logistic model
and several mixture models (similar to DNA content flow cytometry with
two log-normals and a middle quasi-uniform distribution), and found that
the log-normal robustly fit well. The log-normal and logistic largely
agreed on ranking of estimated means, but the likelihood was slightly
higher for the log-normal fits on the whole library, so we used that
model.

From this model (fit using \texttt{mle()} in R), we used the fit 
mean as the estimate for the GAP1 abundance for that strain in that
sample.

\section{Discussion}

Regulated changes in mRNA stability allow cells to rapidly reprogram
gene expression, clearing extant transcripts that are no longer
required and potentially reallocating translational capacity.
%Despite progress in understanding the pathways that mediate
%mRNA degradation, the functional role of mRNA degradation and the
%factors that control regulated changes in mRNA stability remain poorly
%understood. 
Pioneering work in budding yeast has shown that mRNA
stability changes facilitate transcriptome reprogramming in response to
changes in nutrient availability including changes in carbon sources
\parencite{scheffler1998control} and iron starvation
\parencite{puig2005coordinated}. 
Here, we characterized genome-wide changes
in mRNA stability in response to changes in nitrogen availability and
identified factors that mediate the rapid repression of the
destabilized mRNA, \textit{GAP1}. Our study extends our previous work
characterizing the dynamics of transcriptome changes using chemostat
cultures \parencite{airoldi2016steady} and shows that accelerated mRNA
degradation targets a specific subset of the transcriptome in response
to changes in nitrogen availability. We developed a novel approach to
identify regulators of mRNA abundance using pooled mutant screens and
find that modulators of decapping activity, and core degradation
factors, are required for accelerated degradation of 
\textit{GAP1} mRNA. 
 
Measuring the stability of the transcriptome requires the ability to
separate pre-existing and newly synthesized transcripts. We modified
existing methods to measure 
post-transcriptional regulation of the yeast transcriptome in a
nitrogen upshift using 4-thiouracil labeling
\parencite{miller2011dynamic,neymotin2014determination,munchel2011dynamic}. These
modifications included improved normalization and quantification of
extant transcripts and explicit modeling of labelling dynamics to
account for some of the inherent limitations of metabolic labeling
approaches \parencite{perez2013eukaryotic}. Continued development of
fractionation biochemistry \parencite{duffy2015tracking} and incorporation of
explicit per-transcript efficiency terms will improve these
methods further \parencite{chan2017non}.

Our experiments show that the process of physiological and 
transcriptome reprogramming occur on very different timescales in response
to a nitrogen upshift. Cellular physiology is remodeled over the
course of two hours to achieve a new growth rate.
By contrast, transcriptome remodeling occurs rapidly and through
states that are distinct
from increases in steady-state growth rates. 
%Interestingly,
%we found that the yeast transcriptome is on average less stable in
%using metabolic labeling
%\parencite{Munchel2011,Neymotin2014,Miller2011}. This relative
%reduction in mRNA stability could be an adaptation to potentially
%limiting ribonucleotides, but further work exploring differences in
%mRNA degradation rates during growth limited by different nutrients is
%required to test this concept \parencite{Garcia-Martinez2016}.
%Stability changes upon the nitrogen upshift generally exhibited the
%expected relationship with rates of abundance change (
%anti-correlation, $R^2=$-0.376 ); however, we found multiple cases in
%which increased mRNA degradation rates did  not result in rapid
%decreases in mRNA abundance. This has been observed for transcripts
%up-regulated in stress conditions, and has been proposed as a
%mechanism to effect a rebalancing of the transcriptome after a
%transient phase of reprogramming \parencite{Shalem2008}. Importantly,
%the changes in mRNA stability that we detect are nearly coincident
%with the environmental perturbation suggesting that a signal is sensed
%and the effect propagated to impact post-transcriptional regulation
%with rapid kinetics.
Previous studies have shown that transcriptional activation of the NCR
regulon is rapidly repressed upon a nitrogen upshift
\parencite{airoldi2016steady}. Our
results indicate that accelerated degradation of 
%at least 16 of the 77 probable 
many NCR transcripts \parencite{godard2007effect} contributes to this
repression. 
A three-fold increase in
the degradation rate of \textit{GAP1} mRNA provides an additional layer of
repressive control. Importantly, our results show that accelerated
degradation is not limited to NCR transcripts but also targets
transcripts enriched in carbon metabolism pathways, particularly
pyruvate metabolism. Conversely, we also detect an apparent reduction in the 
degradation rate for some transcripts 
%enriched in ribosome biogenesis mRNAs (for example \textit{NSR1}) as well as 
including \textit{MAE1}. \textit{MAE1} encodes
an enzyme responsible for the conversion of malate to pyruvate, and
combined with the accelerated degradation of \textit{PYK2} mRNA 
may reflect an adaptive shunt of carbon skeletons from glutamine 
to alanine via the TCA cycle \parencite{boles1998identification}. 
%Due to the limitations of
%labeling approaches \parencite{Perez-Ortin2013} we cannot conclude 
%here that these transcripts are indeed stabilized, 
%however we can conclude that they are strongly upregulated. 
Recently, \cite{tesniere2017relief}
described destabilization  of carbon metabolism mRNAs after repletion
of nitrogen following 16 hours of starvation. We do
not detect destabilization of \textit{PGK1} mRNA and note that
the basal half-life of 6.2 minutes estimated in our study is similar
to the accelerated rate reported by \cite{tesniere2017relief}.

%To identify the factors that underlie accelerated mRNA degradation, we
%developed a global \textit{trans}-factor screen using mRNA FISH, FACS, and
%sequencing. 
%BFF identified mutants in the Lsm1-7p/Pat1p
%complex as having elevated \textit{GAP1} mRNA levels both before and after the
%upshift.
%Given that the \textit{GAP1} mRNA is destabilized during this
%transition we suspect that these core mRNA degradation factors are
%directly involved. 
%Because factors associated with the Lsm1-7p/Pat1p
%complex are also involved in processing-body formation we looked for
%processing-body dynamics during the nitrogen upshift, but did not see
%qualitative changes in Dcp2-GFP distribution (raw data available in
%supplement). However, it has been proposed that pre-existing mRNPs
%seed the formation of processing-bodies \parencite{Lui2014}, thus the
%phenotype may require assays at a finer spatial scale to eliminate
%this possibility \parencite{Rao2017}. Interestingly, during
%cross-comparisons with a recent dataset exploring mRNA localization to
%RNP condensates \parencite{Khong2017} we found that the set of
%destabilized transcripts in the label-chase experiment are on average
%longer in CDS and have an increased codon-optimality, two factors that
%were shown to be associated with differences in stress-granule
%localization of mRNA \parencite{Khong2017}. 

%Regulated changes in mRNA stability can be mediated by RBP binding the
%3' UTR of specific transcripts.  However, cis element analysis are
%inconsistent with a role for known RBPs in the observed
%destabilization. In particular, Puf3p motifs are de-enriched from the
%destabilized set. We failed to detect new 3' UTR sequence motifs that
%are enriched in destabilized transcripts, but these 
Destabilized
transcripts are enriched for a binding motif of Hrp1p in
the 5' UTR. This essential component of mRNA cleavage for
poly-adenylation in the nucleus has been shown to shuttle to the
cytoplasm and bind to amino-acid metabolism mRNAs
\parencite{guisbert2005functional} and been shown to interact genetically to
mediate nonsense-mediated decay (NMD) of a \textit{PGK1} mRNA harboring a
premature stop-codon \parencite{gonzalez2000yeast} or a \textit{cis}-element spanning
the 5' UTR and first 92 coding bp of \textit{PPR1} mRNA
\parencite{kebaara2003upf}.
A potential role for these Hrp1p sites warrants further investigation. 

BFF identified mutants in the Lsm1-7p/Pat1p
complex as having elevated \textit{GAP1} mRNA levels both before and after the
upshift. This is expected given their central role in mRNA 
metabolism, and experiments using \textit{GAP1} normalized to
\textit{HTA1} demonstrate that the effect before the upshift is
likely a global effect (\autoref{fig:figure5qpcr}). 
However, these mutants still have a
significant defect in clearance of \textit{GAP1},
and the assay demonstrates that associated decapping factors 
\textit{EDC} and \textit{SCD6} have specific effects
(\autoref{fig:figure5qpcr}).
Given that the \textit{GAP1} mRNA is destabilized during this
transition we suspect that these mRNA degradation factors are
directly involved. 
While we found that the \textit{edc3}$\Delta$ mutant has defects in
clearance of \textit{GAP1}, we also 
found that \textit{scd6}$\Delta$,
%mutant shares a phenotype of reduced \textit{GAP1} mRNA
% during nitrogen limitation and reduced rate of \textit{GAP1} mRNA
%clearance with a 
\textit{tif4632}$\Delta$, and deletion of the 5' UTR
of \textit{GAP1} impairs clearance (\autoref{fig:figure5qpcr}). 
This deletion does not include the TATA box (ending at -179) or
GATAA sites (nearest at -237) responsible for NCR GATA-factor
regulation of \textit{GAP1} \parencite{stanbrough1996two}.
This suggests that interactions of
these factors with \textit{cis}-elements in the 5' UTR might be responsible for
stabilizing \textit{GAP1} mRNA during limitation, although the 
truncation of the 5' sequence may be enough to inhibit translation 
initiation by virtue of the shorter length
\parencite{arribere2013roles}.
Elements in the 5' UTR have
also been demonstrated to modulate \textit{GAL1} mRNA stability
\parencite{baumgartner2011antagonistic} and destabilize \textit{SDH2} mRNA upon glucose
addition, perhaps due to the competition between translation
initiation and decapping mechanisms \parencite{de2002role}.
Interestingly, both \textit{GAP1} and \textit{SDH2} 
share the feature of a second start
codon downstream of the canonical start
\parencite{neymotin2016multiple} and
we have previously found that mutation of
the start codon of \textit{GAP1} results in lower
steady-state mRNA abundances \parencite{neymotin2016multiple}.
This
%in light of recent analyses further highlighting the contribution of
%translation dynamics to mRNA stability 
%\parencite{Presnyak2015,Neymotin2016,Cheng2017}, 
suggests a mechanism of degradation through dynamic changes in 
translation initiation that triggers decapping of \textit{GAP1} 
and other mRNA. 
%However, the deletion of \textit{SCD6} would be
%expected to promote translation of mRNA on the basis of it's measured
%repressive activity in cell extracts, suggesting that if Scd6p does play a role
%that it may be specified by some condition-specific modulation of its
%activity \parencite{Rajyaguru2012,Poornima2016}. 
Future work interrogating
this possible interaction of translational status and mRNA
stability during dynamic conditions could also expand our understanding of
the relationship between these two processes.

To our knowledge, this is the first time mRNA abundance has
been directly estimated using a SortSeq approach, although 
%sorting on indirect markers or 
using mRNA FISH and FACS to enrich subpopulations of cells has been
previously reported
\parencite{klemm2014transcriptional,hanley2013detection,sliva2016barcode}. This
approach could be used with other barcoding mutagenesis technologies,
like transposon-sequencing or Cas9 mediated perturbations, to
systematically test the genetic basis of transcript dynamics.
%phenotypes. A strategy combining this technology with transcriptomics
%as a high-dimensional marker could accelerate unbiased investigation
%of cellular signalling pathways \parencite{Gapp2016}. Additionally, 
The use of branched-DNA mRNA FISH, or other methods
\parencite{rouhanifard2017single}, allows for mRNA abundance estimation without
requiring genetic manipulation which makes it suitable for a variety
of applications such as extreme QTL mapping. 
%While the cell wall
%of yeast makes optimization crucial to this assay, future development
%of hybridization protocols may improve accuracy and make the assay
%more robust \parencite{Richter2017,Wadsworth2017}. 
Furthermore, our methods for library construction should permit accurate
quantification of pooled barcode libraries with small inputs, 
expanding the possibilities for flow cytometry markers to fixed-cell assays.

Why is \textit{GAP1} subject to multiple layers of gene product repression upon
a nitrogen upshift, at the level of transcript synthesis, degradation,
protein maturation, and post-translational inactivation? Given the
strong fitness cost of inappropriate activity
\parencite{risinger2006activity},
this overlap could ensure mechanistic redundancy for robust repression in
the face of phenotypic or genotypic variation. Alternatively, it could
reflect a systematic need to free ribonucleotides or
translational capacity, or result from some as yet uncharacterized
process.
%, or could simply be an effect of some unrelated function. 
%While this question remains open,
%we have made progress towards this goal by identifying 
%factors required for its accelerated degradation.
%of decapping associated with the Lsm1-7p/Pat1p complex play a role. 
Future work aimed at determining the adaptive basis of accelerated
mRNA degradation will serve to illuminate the functional role of
post-transcriptional mRNA abundance regulation.
%dissecting thisb
%mechanism and contrasting the dynamic process of mRNA destabilization
%during other growth transitions would greatly inform our understanding
%of mRNA stability specification at steady-state, possibly in light of
%the relationship between translation and stability of mRNAs. 

\section{General methods and materials}

\subsubsection{Availability of data and analysis scripts}

\label{subsubsection:codeanddata}

Computer scripts used for all analyses are available as a git repository
on GitHub\\
(\url{https://github.com/darachm/millerBrandtGresham2018})
and data is available as zip archives on the Open Science
Framework (\url{https://osf.io/7ybsh/}).

To reproduce the entire analysis, or to access a particular 
analysis, clone the git repo.  
Download the \texttt{zip} data archives from the above
OSF link, and put them inside this git repo folder 
(here, \texttt{millerBrandtGresham2018}).
At minimum, you should have the \texttt{data.zip} archive in that directory,
although records of all R analyses are in \texttt{html\_reports.zip}
and intermediate files are in \texttt{tmp.zip}.
Consult the \texttt{README.md} file in the repository for more instructions
and options, including how to unzip intermediate files and HTML
reports generated for every R script which detail the results.

A Shiny application is also available to explore the two main 
datasets in this paper more easily, at
\url{http://shiny.bio.nyu.edu/users/dhm267/}. It
is also included in the OSF as a separate zipped archive for local
installation and long-term archiving. 
To use the Shiny applications from the zipped archive, download,
unzip the archive, and direct R to run the \texttt{runApp} command
to use the directory as a Shiny app. It can thus be used as an
interactive tool for visualization.

\subsubsection{Supplementary tables}

These are available on the Open Science Framework archive
\url{https://osf.io/9ct3m/} .

\begin{itemize}
  \OnehalfSpacing
  \setlength\itemsep{0em}
  \item Gene set enrichment analysis of loadings on principal 
    components one and two.
    \\\texttt{Figure1\_Table\_GSEofGOtermsAgainstPCcorrelation.csv}
    \label{itm:microarrayPCAgsea}
  \item Raw counts of labeled mRNA quantified by RNAseq in 
    label-chase experiment.
    \\\texttt{Figure2\_Table\_RawCountsTableForPulseChase.csv}
    \label{itm:dme211raw}
  \item Filtered label-chase RNAseq data for modeling, normalized 
    directly within sample.
    \\\texttt{Figure2\_Table\_PulseChaseDataNormalizedDirectAndFiltered.csv}
    \label{itm:dme211filterDirect}
  \item Filtered label-chase RNAseq data for modeling, normalized by
    modeling across samples.
    \\\texttt{Figure2\_Table\_PulseChaseDataNormalizedByModel.csv}
    \label{itm:dme211filterModel}
  \item Degradation rate modeling results, from data normalized 
    within samples.
    \\\texttt{Figure2\_Table\_PulseChaseModelingResultTable\_DirectNormalization.csv}
    \label{itm:dme211resultsDirect}
  \item Degradation rate modeling results, from data normalized 
    across samples.
    \\\texttt{Figure2\_Table\_PulseChaseModelingResultTable\_ModelNormalization.csv}
    \label{itm:dme211resultsModel}
  \item Enriched GO and KEGG terms within the set of mRNA 
    destabilized upon a nitrogen upshift, across sample normalization.
    \\\texttt{Figure2\_Table\_AcceleratedDegradationTranscripts\_EnrichedGOandKEGGterms.csv}
    \label{itm:dme211goAndKegg}
  \item Raw counts of strain barcode quantification within each bin
    in the BFF experiment, and gate settings for the observations.
    \\\texttt{Figure4\_Table\_BFFcountsAndGateSettingsFACS.csv}
    \label{itm:dme209rawCountsGates}
  \item BFF data filtered for modeling.
    \\\texttt{Figure4\_Table\_BFFmodelingData.csv}
    \label{itm:dme209modelData}
  \item The parameters of all models fit to the BFF data.
    \\\texttt{Figure4\_Table\_BFFallFitModels.csv}
    \label{itm:dme209allFits}
  \item All 3230 models used for identifying strains with defective 
    \textit{GAP1} dynamics.
    \\\texttt{Figure4\_Table\_BFFfilteredPooledModels.csv}
    \label{itm:dme209pooledFits}
  \item Gene-set enrichment analysis results using \textit{GAP1} 
    estimates.
    \\\texttt{Figure4\_Table\_GSEanalysisOfBFFresults.csv}
    \label{itm:dme209gsea}
\end{itemize}

\subsubsection{Media and upshifts of media}

Nitrogen-limited media (abbreviated as "Nlim") is a minimal media
supplemented with various salts, metals, minerals, vitamins, and
2\% glucose, as previously described
\parencite{airoldi2016steady,brauer2008coordination}. 
For proline limitation, 
Nlim base media was made with 800$\mu$M L-proline as the sole
nitrogen source (NLim-Pro).
YPD media was made using standard recipes \parencite{amberg2005methods}.
All growth was at 30$^{\circ}$C, in an air-incubated 200rpm shaker  
using baffled flasks with foil caps, or roller drums for 
overnight cultures in test tubes.
For glutamine upshift experiments, 
400$\mu$M L-glutamine was added from a 100mM stock solution dissolved 
in MilliQ double-deionized water and filter sterilized.
All upshift experiments were
performed at a cell density between 1 and 5 million cells per mL,
in media where saturation is approximately 30 million cells per mL. 
For all experiments, 
a colony was picked from a YPD plate and grown in a 5mL NLimPro 
pre-culture overnight at 30$^{\circ}$C, then innoculated into
the experimental culture from mid-exponential phase.

\subsubsection{Strains}

See \autoref{tab:strainsTable} for details.  
The wild-type strain used is FY4, a S288C derivative. 
The pooled deletion collection is as published in 
\cite{vandersluis2014broad}.
For all experiments with single strains, colonies were struck 
from a -80$^{\circ}$C frozen stock onto YPD (or YPD+G418 for
deletion strains) to isolate single colonies.
For pooled experiments we inoculated directly into NLim media
from aliquots of frozen glycerol stocks.

\begin{table}%[bt]
\caption{Yeast strains used in this study}
% Use "S" column identifier to align on decimal point 
\label{tab:strainsTable}
\begin{tabular}{l l p{.5\textwidth}}
\toprule
Strain ID & Short description & Details \\
\midrule
DGY1 & FY4 & Isogenic to S288C, prototrophic, MATa \\
- & Deletion collection pool & Haploid (MATa) prototrophic deletion
collection as described in the publication of
\cite{vandersluis2014broad}\\
DGY410 &xrn1$\Delta$::KanMX &   ygl173c$\Delta$::KanMX from the prototrophic deletion collection \\
DGY564 &ccr4$\Delta$::KanMX &   yal021c$\Delta$::KanMX from the prototrophic deletion collection \\
DGY565 &pop2$\Delta$::KanMX &   ynr052c$\Delta$::KanMX from the prototrophic deletion collection \\
DGY547 &lsm1$\Delta$::KanMX &   yjl124c$\Delta$::KanMX from the prototrophic deletion collection \\
DGY571 &lsm6$\Delta$::KanMX &   ydr378c$\Delta$::KanMX from the prototrophic deletion collection \\
DGY545 &pat1$\Delta$::KanMX &   ycr077c$\Delta$::KanMX from the prototrophic deletion collection \\
DGY554 &edc3$\Delta$::KanMX &   yel015w$\Delta$::KanMX from the prototrophic deletion collection \\
DGY552 &scd6$\Delta$::KanMX &   ypr129w$\Delta$::KanMX from the prototrophic deletion collection \\
DGY611 &tif4632$\Delta$::KanMX &   ygl049c$\Delta$::KanMX from the prototrophic deletion collection \\
DGY539 & \textit{GAP1} 5' UTR delete & confirmed by Sanger sequencing to have 152bp deleted 5' of the start codon \\
DGY576 & \textit{GAP1} 5' UTR delete & confirmed by Sanger sequencing to have 100bp deleted 5' of the start codon \\
DGY577 & \textit{GAP1} 3' UTR delete & confirmed by Sanger sequencing to have 150bp deleted 3' of the stop codon \\
DGY525 & FY3 + pRP1315 & FY3, a ura- auxotroph (ura3-52), transformed with pRP1315 (URA3 marker, expressing a Dcp2-GFP fusion) \\
\bottomrule
\end{tabular}
\end{table}

\begin{table}%[bt]
\caption{Primers used in this study}
\label{tab:primerTable}
% Use "S" column identifier to align on decimal point 
\begin{tabular}{p{.1\textwidth} l p{.25\textwidth}}
\toprule
ID & Sequence & Description \\
\midrule
DGO230 & \scriptsize\ttfamily ACGGTATCAAGGGTTTGCCAAG & Figure 3 qPCR \textit{GAP1} reverse \\
DGO232 & \scriptsize\ttfamily GCATAAATGGCAGAGTTAC & Figure 3 qPCR \textit{GAP1} forward \\
DGO229 & \scriptsize\ttfamily CTCTACGGATTCACTGGCAGCA & Figure 5 qPCR \textit{GAP1} reverse \\
DGO231 & \scriptsize\ttfamily TTTGTTCTGTCTTCGTCAC & Figure 5 qPCR \textit{GAP1} forward \\
DGO236 & \scriptsize\ttfamily TTACCCAATAGCTTGTTCAATT & qPCR HTA1 forward  \\
DGO233 & \scriptsize\ttfamily GCTGGTAATGCTGCTAGGGATA & qPCR HTA1 reverse  \\
DGO605 & \scriptsize\ttfamily CTGGACGACTTCGACTACGG & qPCR 1200 spike-in forward \\
DGO606 & \scriptsize\ttfamily ATCAGCCTTTCCTTTCGTCA & qPCR 1200 spike-in reverse \\
DGO1562 & \scriptsize\ttfamily
GTCTGAACTCCAGTCACATCNCNCNCNTNCNGTCGACCTGCAGCGTA & Degenerate first round primer \\
DGO1588 & \scriptsize\ttfamily CCATTGGTGAGCAGCGAAGGATTTGGTGGA/3Phos/ & First round blocker oligo \\
DGO1589 & \scriptsize\ttfamily AGAAAAAGCAGCGTAGATGTAGAAGCAAGA/3Phos/ & First round blocker oligo \\
DGO1567 & \scriptsize\ttfamily GATGTCCACGAGGTCTCT & Second round outside primer \\
DGO1576 & \scriptsize\ttfamily CGTACGCTGCAGGTCGAC/3Phos/ & Second round blocker oligo \\
DGO1519 & \scriptsize\ttfamily CAAGCAGAAGACGGCATACGAGATGTCTGAACTCCAGTCAC & Second and third round inside primer and P7 adapter \\
Forward index primer & \scriptsize\ttfamily
ACGCTCTTCCGATCTXXXXXGTCCACGAGGTCTCT & Multiplexing primer, where 
XXXXX is one of 120 different barcodes (see below).
\autoref{tab:indexbarcodes}.\\
DGO276 & \scriptsize\ttfamily AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT & Illumina P5 adapter incorporation primer \\
DGO366 & \scriptsize\ttfamily AATGATACGGCGACCACCGAGATCTACAC & RNAseq Illumina library amplification, forward \\
DGO367 & \scriptsize\ttfamily CAAGCAGAAGACGGCATACGAGAT & RNAseq Illumina library amplifcation, reverse \\
\bottomrule
\end{tabular}
\end{table}

All primers were synthesized by Integrated DNA Technologies (IDT).
\texttt{N} refers to a standard degenerate position.
\label{tab:indexbarcodes}
Barseq multiplexing barcode sequences and index numbers available in 
the file 
\texttt{data/dme209/sampleBarcodesRobinson2014.txt} within the
data zip archive (\nameref{subsubsection:codeanddata}). \\

